
```{r CH4--chunkopts, echo = FALSE}

knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE,
                      background = "#707070", fig.align = "center", cache = FALSE)

#options(knitr.table.format = "html")
#options(knitr.table.format = "latex")

word <- TRUE

options(knitr.graphics.auto_pdf = TRUE)

```

```{r CH4--setup, echo=FALSE}

## This chapter .Rmd code relies on two sourced R codes:
## 'fungi_data_processing.R' and 'fungi_data_analysis.R.' These do the heavy
## lifting before importing results here for ggplot-ifying and presenting in
## Results and Discussion.

## Data manipulation --> analysis  --> Rmarkdown (this doc)


## Load sourced R codes
all.current <- TRUE

if (all.current == TRUE) {

    load(file = "../Fungi/fungi_data_analysis_complete_09072018.RData")

}

if (all.current == FALSE) {
    # This takes a few hours.
    source("../Fungi/fungi_data_analysis.R")
    
}

setwd("../DissertationBookdown/")

## Load required libraries
    library(ggplot2)
    library(rgl)
    library(pander)
    library(gridExtra)
    library(ggrepel)
    library(tibble)
    library(stringr)
    library(vegan)
    library(knitr)
    library(kableExtra)
    library(TITAN2)
    library(data.table)
    #library(huxtable)
    library(dplyr)


    options(digits = 3)
    options(scipen = 9999)
    
    source("../R_Scripts/PlotTaxaKD.R")
    
    pretty.labs <- function(x) {str_replace(x, pattern = "__", replacement = "/") %>%
                                    str_replace_all(pattern = "_", " ")}



```

# Fungal Communities on Office Developments {#CH4-start}

## Introduction {#CH4-introduction}

Human actions contributing to novel ecosystem processes, including actions taken during development and land management that contribute to disturbance and succession processes, impact vegetation communities on office developments and may also impact urban fungal communities. Human actions during development contribute to native conifer loss, soil disturbance and compaction, and impervious surface creation. Land management, comprising landscaping and maintenance actions, changes tree and shrub community composition, host species availability, and alters nutrient cycling processes through the addition of water, fertilizer, and mulch.


These human actions may alter habitat quantity, soil nutrient availability, and the host species and substrates available to fungi. In non-urban settings, fungal community composition is sensitive to changes in the availability of host plants, soil nutrients, and irrigation, though different fungal taxa and trophic groups may respond differently to the same variable [@batten2006; @lothamer2014; @lilleskov2002; @schimel2007; @wiklund1995]. For example, the addition of fertilizer creates an influx of nutrients that may reduce fungal species richness with a disparate impact on ectomycorrhizal fungi [@lilleskov2002; @avis2008]. 


Changes in the fungal community caused by altering environmental conditions may also alter the ecosystem functions delivered by fungal communities [@vanderheijden1998; @zak2003]. Many of these ecosystem functions are driven by how fungi acquire carbon---that is, their trophic guild. For example, saprotrophic fungi decompose wood and litter via a diverse set of enzymatic pathways and are important wood and litter decomposers---an ecosystem process that recycles complex organic matter but also contributes to degradation of human infrastructure [@blanchette2000; @isaacs2016; @rabinovich2004; @martinez2005]. Symbiotrophic fungi, including mycorrhizal fungi and endophytic fungi, form partnerships with plant species, enhancing mineral nutrient and water acquisition by plant species and influencing plant community composition [@jeffries2003; @brundrett1991; @lehto2011]. Pathotrophic fungi are dependent on a host organism (plant, animal, fungi, bacteria, algae) for resources, and in urban areas infect plants, urban trees, and (more rarely) humans [@schubert2008]. This is a simplified view---many fungal taxa belong to multiple trophic guilds, such as pathotrophic taxa which access the dead wood of live trees and weaken them to the point where they are susceptible to wind throw or other pathogens---however, it provides a useful framework for evaluating changes in ecosystem proceses and service provision.   

Existing research on urban fungi and changes in fungal community composition driven by novel ecosystem processes is sparse [@newbound2010; @newbound2012; @pautasso2013]. Existing studies generally examine agricultural or remnant ecosystems, and/or address individual impacts of novel ecosystem process [e.g. fertilizer only; @mcdonnell1997; @newbound2012]. The impacts of other important human actions contributing to novel ecosystem processes have not yet been addressed.

In this chapter, I compare measures of the fungal community observed on office development sites---including effective species richness, community composition, and trophic classification---with variables describing the outcome of human action during development and land management. I address the following questions regarding fungal communities and functional groups:


1. What fungal taxa are present on office development sites?
2. Do variables describing neighborhood scale socio-economic or land cover and/or variables describing the outcome of development, landscaping, or management actions explain variation in fungal or trophic group effective species richness?
3. Do variables describing neighborhood scale socio-economic or land cover and/or variables describing the outcome of development, landscaping, or management actions explain variation in fungal community composition or trophic group community composition?
4. Is there evidence for community thresholds in fungal communities for significant environmental gradients identified in Q3?
	
	
I hypothesized that site scale variables would be more important than neighborhood scale variables following previous research [@newbound2012] and that variables describing woody debris availability would be significant in explaining variation in fungal species richness and fungal community composition for the whole community as well as saprotrophic and symbiotrophic subsets. I also hypothesized that sites where existing trees and soils have not been conserved will have more saprotrophic taxa richness, and reduced symbiotrophic taxa richness.

To address these questions, I quantified the fungal communities on office development sites using next-generation sequencing (NGS) data from soil samples taken from 17 office development sites and 2 adjacent forest parcels. I also collected fungal sporocarp (mushroom) specimens from 19 office development sites. I quantified soil nutrients, tree and shrub communities, and ground cover proportions, neighborhood and site scale socio-economic, neighborhood scale land cover variables, and variables describing the outcome of development, landscaping, and maintenance actions for each site. I used univariate and multivariate ecology tools to analyze NGS and collected fungal specimens datasets.
To address these questions, I quantified the fungal communities on office development sites using next-generation sequencing (NGS) data from soil samples taken from 17 office development sites and 2 adjacent forest parcels. I also collected fungal mushroom specimens from 19 office development sites. I quantified soil nutrients, tree and shrub communities, and ground cover proportions, neighborhood and site scale socio-economic, neighborhood scale land cover variables, and variables describing the outcome of development, landscaping, and maintenance actions for each site. I used univariate and multivariate ecology tools to analyze NGS and collected mushroom datasets.

My results suggest that species richness is largely uniform across office development sites, while fungal community composition and trophic guild community composition are significantly influenced by development, landscape design and maintenance actions taken by developers and land owners. Actions impacting tree preservation through development, tree and shrub community composition, woody debris availability, and soil chemistry are consistently important, with significant variation explained in PERMANOVA and change points detected in TITAN. However, the relatively small effect size in PERMANOVA suggests a need for further study to verify these results.

```{r fungi-conceptual-diagram, fig.cap='(ref:fungiconceptual)', out.width='90%'}

	knitr::include_graphics("images/FungiVariables2.png", auto_pdf = TRUE)


```

(ref:fungiconceptual) An overview of the relationships between measured independent variables---including vegetation discussed in [Chapter 2](#CH2-introduction)---and the fungal community present at a site. The site context includes: historical vegetation for the region; abiotic factors including site hydrology, area, and location; regional/neighborhood land cover patterns which contribute to the habitat matrix exploited by bird species; land use policy including tree protection policies and building and pavement surface area limitations; and socio-economic variables including median income and area demographics. When a site is developed, the developer’s actions determine whether stands of trees and their surrounding soil and root fungal communities are preserved through development, the initial conifer density (both preserved and planted), the initial planting design (including native vs. ornamental trees and shrubs), and the year built. These variables are further modified by the land management actions, including herbicide/pesticide/etc. applications, dead wood removal, and plant replacement (replacing dead plants or refreshing the planting design). These variables may influence fungal community composition.  For example, vegetation on site controls the partner or host species available for mycorrhizal and pathogenic fungi. Litter from these species as well as availability of deadwood provide substrate for saprotrophic fungal taxa. Historical vegetation and fungal presence on the site combined with soil/stand preservation, fertilization/nutrient availability, pH and other soil conditions may also influence what fungal species can survive on the site. Further, a subset of the site context variables---such as the socio-economic and land cover variables---may influence the surrounding fungal community and therefore control the fungal community at the neighborhood scale and species available to disperse onto the site.





## Literature review {#CH4-literaturereview}

The existing fungal literature  focuses on forested, grassland, and other natural ecosystems, while a limited number of studies address urban fungi [@newbound2010]. However, these studies provide context for the variable relationships described in Figure \@ref(fig:fungi-conceptual-diagram). In this review, I provide a summary of fungi and their ecosystem function and discuss the current state of the urban fungal literature, before describing potential impacts of novel ecosystem processes driven by development and land management actions described in Figure \@ref(fig:fungi-conceptual-diagram). 

### Fungi and ecosystem function

Fungi perform a diverse set of ecosystem functions, depending on a species’ life history characteristics. The four major fungal guilds are saprotrophic (decomposers), mycorrhizal, parasitic, and endophytic fungi. Despite their prevalence, many parasitic and endophytic fungi rarely if ever produce sporocarps and thus can only be detected using rRNA [@rodriguez2009]. Many saprotrophic fungi are molds, yeasts and other microscopic fungi that can only be directly viewed by growing them in culture. Fungal species frequently fulfill multiple roles—for example, mycorrhizal species may also have some saprobic abilities [@brundrett1991; @lindahl2002; @lindahl2015; @shah2016]. 

#### Decomposition: nutrient regulation {-}

Decomposition of litter and wood is a vital ecosystem process that recycles complex organic matter [@blanchette2000]. Fungal taxa can decompose recalcitrant organic matter via hyphal growth that can penetrate solid material and extracellular enzymes that degrade lignocellulose [@vanderwal2013]. Fungi employ multiple enzymatic pathways for decomposition, the most common of which are white rot, brown rot, and soft rot [though this paradigm may be inadequate, see @riley2014]. White rot fungi readily degrade lignin as well as pectin, cellulose, and hemicellulose. White rot fungi include basidiomycetes and some ascomycetes and more frequently decompose hardwoods than conifers. Brown rot fungi degrade cellulose and hemicellulose while leaving the lignin mostly intact. Brown rot fungi are typically basidiomycetes and primarily decompose conifers [@martinez2005; @rabinovich2004; @blanchette1991; @floudas2012]. Soft rot fungi break down cellulose and hemicellulose, and while less aggressive than brown- and white-rot fungi are able to thrive in adverse chemical and thermal environments [@vane2005; @martinez2005]. These fungi are typically ascomycetes and decompose both conifers and hardwood trees [@rabinovich2004]. 

Leaf litter and coarse woody debris provide different environments for decomposition and are exploited by different communities. Leaf litter itself varies significantly; litter may be rich or poor in easily available carbon and nutrients, and may contain additional compounds such as tannins which inhibit decomposers [@gessner2010; @ivarson1959]. Coarse woody debris is comparatively carbon rich and nutrient poor [including N, P, K; @laiho2004]. 

The rate of decomposition varies based on litter or wood quality, composition of the decomposer community, and the physicochemical environment [@hattenschwiler2005; @vanderwal2015; @krishna2017]. Competition between saprotrophic fungi and mycorrhizal fungi species that are also capable of decomposition influences the composition of the decomposer community [@hattenschwiler2005]. For example, some ericoid and ectomycorrhizal fungi can also degrade lignin [@haselwandter1990]. White- and brown-rot fungi differentially influence soil acidity, which in turn contributes to determining what species of mycorrhizal species are present [Ammirati, personal communication, @vane2005]. 

#### Mycorrhizal associations: nutrient regulation, water regulation; indirect contribution to EF provided by plants {-}

Most plant species form mycorrhizal associations---mutualistic associations between plant roots and fungi---with  fungi [@brundrett1991]. Ectomycorrhizal fungi (predominantly _Basidiomycota_ or _Ascomycota_) are sepate (dividers between cells of hyphae), and do not penetrate the host plant's cell walls, instead forming a Hartig net (a lattice of branched hyphae) and a hyphal sheathe around the root surface of _Gymnospermae_ and _Angiospermae_ [@smith2008]. Arbuscular mycorrhizal fungi (_Glomeromycota_) are common, associating with over 80% of all plant species, are asepate, and use intracellular colonization to interface with host plants including _Bryophyta_, _Pteridophyta_, _Gymnospermae_ and _Angiospermae_[@redecker2013; @smith2008]. Ericoid mycorrhizae (_Ascomycota_) are sepate and use intracellular colonization to interface with _Ericales_ and _Bryophyta_ [@smith2008]. Other mycorrhizal types include Ectendomycorrhiza (_Pinus_ and _Larix_ spp.), Arbutoid mycorrhiza (exclusively with _Ericales_), Monotropoid mycorrhiza (exclusive with _Monotropoideae_), and Orchid mycorrhiza (exclusive with _Orchidales_).

Mycorrhizal fungi generally enhance mineral nutrient acquisition, though the different groups transfer different soil nutrients to their symbiotic plants [particularly Phosphorous; @jeffries2003; @vanderheijden2017; @reynolds2005]. Evidence of between plant transport of nutrients is better substantiated for ectomycorrhizal than arbuscular mycorrhizal fungal taxa [@smith2008]. As a result, mycorrhizal fungi play an important role in plant growth and nutrient cycling [@degroot2002; @newsham1995; @swift1979; @hoeksema2010].

Mycorrhizae also influence water regulation via plant uptake, though with more varied results. Arbuscular mycorrhizae often provide their hosts with water [@jeffries2003]. Evidence of water uptake by ectomycorrhizae is less clear, and mostly limited to small seedlings. There is no evidence of ericoid mycorrhizas directly influencing water uptake. However, both ecto- and ericoid mycorrhizae likely influence their host plants' water relations through increased nutrition [@lehto2011]. 

By enhancing nutrient and water acquisition, mycorrhizal fungi may indirectly increase plant resistance to disease, drought, competition, and other stresses [@bennett2006; @lehto2011]. Mycorrhizal and endophytic fungi also trigger plant defenses [@jung2012; @pozo2007]. As a result, mycorrhizal fungi are a major factor influencing plant biodiversity, community composition and attendant ecosystem functions [@hoeksema2010; @brundrett1991; @vanderheijden1998; @dahlberg2001; @read2004; @vanderheijden2007; @vanderheijden2003; @francis1994; @newsham1995]. 

By supporting trees and other plants, mycorrhizae also contribute indirectly to other ecosystem functions. These include gas regulation, climate regulation, disturbance prevention, water regulation, soil retention, nutrient regulation, and habitat provision [@degroot2002; @read2004]. 

#### Parasitic interactions: food web structure, nutrient regulation {-}

Parasitic fungi are dependent on their host organisms for resources (e.g. nutrients and water) and infect other species of fungi[^lobster], living trees and shrubs, as well as invertebrates and vertebrates. Parasitic fungi may or may not kill their host organism. In urban ecosystems, parasitic fungi commonly infect landscaping plants and urban trees and are responsible for many of the diseases of cultivated plants [@moore1959; @mendgen1996]. Urban trees may be more susceptible to parasitic fungi due to frequent pruning [@schubert2008].

Parasites modify food webs and trophic interactions, competition between organisms, and energy and nutrient transfer [@preston2010; @lafferty2006]. For example, fungal parasites impact the primary productivity of their plant hosts, exerting significant top-down control on biomass [@mitchell2003]. The impact of parasitic fungi on the broader urban ecosystem is unknown as humans respond to fungal impacts. On office developments, plant death will likely result in plant removal and replacement by landscaping crew, while diseased plants may be treated with fungicides or fertilizers (Dyson, unpublished data).


[^lobster]: One highly prized edible mushroom, the Lobster mushroom, is actually a mushroom of the genera _Lactarius_ or _Russula_ that has been parasitized by the fungus _Hypomyces lactifluorum_.


#### Endophytic relationships: plant survival {-}

Endophytic fungi colonize internal plant tissues---much as parasitic and some mycorrhizal fungi do---but do not cause harm to their host or involve the same direct mutualistic benefits of mycorrhizal formation[^petrini] [@hyde2008; @petrini1991]. Many, maybe all, plant species host endophytic fungi, including grasses, ferns, and seed plants [@arnold2003; @hyde2008]. The degree of endophytic relationships ranges from obligate to opportunistic, though most endophytes are facultative endophytes. Endophytes can be transmitted vertically via seeds [@hardoim2015].

The ecosystem function of endophytes is not as well understood as that of the other major fungal groups [@hyde2008]. Plant-endophyte interactions are generally studied under controlled conditions, and interactions may differ under field conditions [@hardoim2015]. Some fungal-host pairs seem to form mutualistic relationships where the fungi contribute to host defense against pathogens and insects [e.g. grasses are well known, likely some woody species as well; @arnold2003]. In grasses, where these relationships are well studied, endophytes may contribute to the dominance of specific species and result in reduced species richness [@clay1999]. Invasive species may also benefit from endophytes in seeds; there is evidence of increased germination and seedling growth in _Phragmites_ [@shearin2018]. Other benefits to the plant may include decreased herbivory, enhanced plant growth, and increased drought resistance [@hyde2008]. 


[^petrini]: Note that @petrini1991 includes in his definition of endophytic fungi fungal parasites that are inactive for much of their life cycle. This is another example of how one fungal species might serve multiple simultaneous or consecutive ecosystem functions. Other researchers suggest that endophytes may become saprobes [@hyde2008].


### Urban fungi 

Much of our knowledge of fungi comes from studies in natural and agricultural settings. Mycorrhizal[^recentresearch], endophytic, saprobic and parasitic fungi in urban areas inhabit novel ecosystems, with notably different plant species, environmental, and soil conditions [@newbound2010; @newbound2012]. For mycorhizal, endophytic, and parasitic fungi, the host plants, environmental and soil conditions, and the species of fungi present are likely to influence the successful formation of mycorrhizae and thus the mycorrhizal fungal community [ @brundrett1991]. For decomposers, the availability of appropriate substrates along with the environmental and soil conditions are likely to be most important to species composition.

[^recentresearch]: Most recent research focuses on mycorrhizal fungi.


#### Urban fungi and ornamental (non-native) plantings {-}

Mycorrhizal, parasitic, and endophytic fungi impact non-native plants' establishment, growth, and biotic interactions [@wolfe2005]. Recent research suggests that interactions between ornamental plants and fungi may impact both plant and fungal diversity and community composition, as well as alter properties of urban soils and impact ecosystem function [@batten2006; @yao2006].

Plants modify their rhizosphere environments and leaf litter type thus influencing what fungal communities are present [@yao2006; @lothamer2014; @vivanco2008]. Introducing non-native plant species alters fungal community composition in other ecosystems; mechanisms include modified soil properties, including pH and nitrification rates [@kourtev2002; @kourtev2003; @wolfe2005; @hawkes2005; @ehrenfeld2003]. Fungal community changes may impact broader ecosystem function and mycorrhizal colonization of native plants [@batten2006; @hawkes2006]. Additionally, mycorrhizal fungal inoculum can diminish when host plants are removed, suggesting a positive feedback mechanism between mycorrhizal fungi and plant hosts [@vogelsang2006]. Using native plants in urban areas is important to maintain native fungal communities---and for the continued success of native plants [@lothamer2014; @corbin2012].

The interaction between ornamental plantings and the urban soil fungal community remains an area in need of further research [@lothamer2014; @kourtev2002; @wolfe2005; @hawkes2005].

#### Urban fungi and modified environmental and soil conditions {-}

The urban environment and soil conditions are novel and highly heterogeneous, and resulting fungal communities provide different ecosystem functions when compared with natural areas [@pouyat2010; @mcdonnell1997]. Microclimates in the urban environment are influenced by physical characteristics of the urban environment[^building] and management, resulting in variable heat, moisture, and pollutant loading at a range of scales [@taha1997; @jumpponen2010]. Resulting environmental stresses such as drought experienced by fungi impose physiological costs and may cause shifts in community composition and the allocation and fate of carbon and nitrogen [@schimel2007].


[^building]: E.g. building and impervious surface geometry.


Urban soil characteristics---including heavy metals pollution, modified nitrogen and phosphorous concentrations, and changes in pH---impact fungal growth and community composition. Different fungal groups, particularly saprotrophic and mycorrhizal fungi, respond differently to urban modified soil chemistry characteristics, particularly nitrogen [@hogberg2003; @jumpponen2010; @lindahl2002; @markkola1995; @markkola2006; @treseder2004; @trudell2004]. Human maintenance actions including watering, pest control, and litter removal also impact fungal community composition [@jumpponen2010; @lilleskov2002; @allison2007; @gehring2006]. See also [Impacts of Land Management on Fungi](#CH4-LR-land-management).

Few studies address fungal community composition spatially, across an urban environmental gradient [@mcdonnell2008]. There is some evidence that the distribution of fungi across the urban gradient appears more closely related to site specific soil characteristics stemming from maintenance than landscape scale variables [@newbound2012]. 

Additional urban fungal studies are necessary to better understand the processes and patterns of fungal distribution, and particularly for different fungal groups with important impacts on ecosystem function [@mcdonnell1997; @pavaozuckerman2005].


#### Urban fungi and other impacts {-}

Many questions concerning fungi in urban environments remain unanswered. Topsoil removal and replacement during development[^topsoil] is a particular concern. In natural areas, soil disturbance impacts may include reduction in the number of viable spores, loss of hyphal networks, and deterrence of hyphal growth to new roots. In urban environments, topsoil modifications during development likely destroys the existing fungal community, alters the spore bank present, and alters soil properties. Studies tracking changes in the urban fungal community through and after development would help illuminate the changes experienced by the fungal community. 


[^topsoil]: Development usually involves a 'tabula rasa' approach of clearing the upper few feet of topsoil and replacing it with imported fill after building completion


Fungal dispersal in urban environments is also poorly understood. In natural areas, dispersal occurs via active processes (hyphal growth) or passive processes [wind carried spores; @brundrett1991]. In urban areas, these processes are augmented by human action. Some nurseries and landscaping companies use fungal inoculants, which likely consist of a limited pool of genetic material. Products are advertised online as effective without species information or consideration for the ecosystem into which they are being introduced. For example, websites claim:


* "Our products contain the top types of mycorrhizal fungi spores from USDA and other tests, appropriate for any combination of plant, soil, and climate" [http://bio-organics.com/]
* "Fungi (Mycorrhizae) are absolutely necessary for your garden’s health, their networks are an essential part of any healthy garden soil. They will harmonize with the plant’s root system and greatly expand the surface area of the root mass. They improve most plants’ growth and vigor. They also promote environmental resistance to disease, pests, heat and drought - through their critical role in nutrient cycling, mediating plant stress and protecting against transplant shock. " [http://www.bountifulgardens.org/Fungal-and-Bacterial-Inoculants/products/9/]


Fungal inoculants, particularly those not from local sources, may not perform the desired function, may create invasive species problems, and non-sterile inoculum may introduce saprobes and parasites [@schwartz2006]. Some fungal species are imported as non-natives on nursery rootstock, including _Paxillus involutus_; scientific guidance on detection and removal of non-native fungi is scarce [@dickie2016]. Patterns of urban dispersal will also impact how fungal succession proceeds following development disturbance. Studies examining fungal dispersal, along with genetic testing of potential vectors (such as inoculum) is required to better understand urban fungal dispersal.


### Impacts of land management on fungi {#CH4-LR-land-management}

Land management actions include nutrient addition, pesticide application, irrigation, and changes in woody debris availability. Fungal groups may be impacted by these actions contributing to novel ecosystem process differently based on their ecology and interspecies competition. 

**Fertilizer:** Fertilizer addition increases locally available nutrient concentrations. The most commonly added nutrients include nitrogen, phosphorous, and potassium. Fertilizer may alter existing fungal communities both within and between fungal groups; fungal function may also be impacted.

Nitrogen addition to systems can result in decreased fungal species richness generally and within particular groups such as ectomycorrhizal fungi [Basiodiomycete diversity @allison2007; ectomycorrhizal fungi @lilleskov2002; @avis2008; @eom1999; mycorrhizal fungi generally @wiklund1995]. Reductions in mycorrhizal richness may be partially explained by the specialties of different mycorrhizal groups; ectomycorrizae are efficient at taking up nitrogen and could lose a competitive advantage in nitrogen rich environments [@lehto2011]. Researchers who study these systems have also observed changes in ectomycorrhizal community composition and suggest that nitrogen addition may cause a shift from taxa specialized for nitrogen uptake (low-N conditions) to taxa specialized for phosphorous uptake [high-N, low-P conditions; @lilleskov2002]. However this response is known to be inconsistent both between systems and between genera of fungi and mycorrhizal hosts [@girvan2004; @rillig1998 using NPK fertilizer; @donnison2000 also NPK], and determining causation is limited by detection issues [@avis2008].

Generally, saprotrophic species are less impacted than mycorrhizal species by nutrient addition [@wiklund1995]. Combined with the sensitivity of mycorrhizal fungi to fertilizer additions, nutrient addition and subsequent changes in fungal taxa richness may alter ecosystem process including decomposition, nutrient cycling, and impact plant communities [@lilleskov2002; @eom1999].

**Irrigation:** Irrigation appears to have a stimulating effect on ectomycorrhizal fungi---a generalized increase in fruiting, spore abundance, and species richness is observed in systems with water input [@wiklund1995]. However, watering also alters community composition. Fruiting in some drought-loving species is suppressed by irrigation [@wiklund1995; @gehring2006], and watering results in higher EM fungal colonization and lower AM fungal colonization [@gehring2006]. As with fertilizer addition, saprotrophic species are less impacted than mycorrhizal species, though wetter microclimates enhance the ability for fungi to colonize [@brantley2001; @wiklund1995]. These changes in fungal community composition may alter decomposition and nutrient cycling ecosystem process [@lilleskov2002; @eom1999].

**Insecticides, Fungicides, Herbicides:** Overall, the sensitivity of fungal communities to chemical application may vary by ecosystem, species, length of application, and the specific chemical used. A number of herbicides, pesticides, and fungicides appear to have no impact across multiple ecosystems, though there may be confounding factors masking a response [@bending2007; Table \@ref(tab:CH4-insecticide-table)]. 


``` {r CH4-insecticide-table}

insecticide.table <- tibble(System = c("Forest",
                                       "Forest",
                                       "Agricultural",
                                       "Agricultural",
                                       "Agricultural; Turfgrass",
                                       "Agricultural",
                                       "Agricultural"),
                            `Measured Variable` = c("Hyphal length",
                                                    "Community structure",
                                                    "Diversity and equitability of fungal communities",
                                                    "Community composition",
                                                    "Community structure",
                                                    "Community structure and abundance",
                                                    "Abundance"),
                            `Chemical class` = c("Herbicide",
                                                 "Fungicide",
                                                 "Herbicide; Pesticide; Fungicide",
                                                 "Fungicide",
                                                 "Fungicide",
                                                 "Herbicide",
                                                 "Herbicide"),
                            `Chemical name` = c("Glyphosate",
                                                "Chlorothalonil",
                                                "Ally/metsulfuron; Marathon/imidacloprid; Folicur/tebuconazole; Amistar/azoxystrobin",
                                                "Captan",
                                                "Chlorothalonil",
                                                "Glyphosate",
                                                "Vision/glyphosate; Release/triclopyr"),
                            Citation = c("@ratcliff2006",
                                         "@sigler2002",
                                         "@girvan2004",
                                         "@beare1993",
                                         "@sigler2002",
                                         "@hart2009",
                                         "@houston1998")
)

knitr::kable(insecticide.table,
	             booktabs = TRUE, caption = "No significantly significant observed impact of chemical on fungal community variable. Note that these studies are field studies only, as artificial media assays are of limited relevance[@busse2001]. All compounds are currently in use.")



```


However, a few chemicals consistently impact fungal communities. Benomyl, a fungicide, reduces mycorrhizal formation, root colonization, and phosphorous transfer [@brundrett1991; @smith2000; @newsham1995; @helgason2007]. However, the compound is no longer in use as of 2001[^benomyl]. Other compounds that impact fungal communities include carbofuran [a pesticide now widely banned[^carbofuran]; @stanton1981], urea, and Rotstop [a biological control agent consisting of _Phlebiopsis gigantea_ spores, @vasiliauskas2004].


[^benomyl]: See http://pmep.cce.cornell.edu/profiles/fung-nemat/aceticacid-etridiazole/benomyl/benomyl_receipt_501.html
[^carbofuran]: See http://www.ens-newswire.com/ens/may2009/2009-05-11-093.asp


**Mulch:** Decomposer fungi feed on dead organic matter. Mulch addition should alter the composition of organic matter available to fungi and thus the species composition of this functional group. Explorations of fungal communities in landscape mulch have found a wide variety of fungal decomposers present [@brantley2001; @hooverlitty1985; @shaw2004]. However, our knowledge about the impacts of mulch addition on these fungal communities is limited [@brantley2001; @shaw2004]. 

There is evidence that the addition of mulch impacts existing fungal communities, stimulating growth and fruiting in species already present in the soil [@bridge2007]. Mulch is also acidic, and pH is known to influence which species are present [@chalkerscott2007]. 

The varied physical properties of mulch added also appears to influence fungal communities. Bark, woodchips, and sawdust support different species diversity [@shaw2004]. Mulch formulations that retain water are easier to colonize, while large nuggets are more difficult to colonize as they repel water [as does any mulch with hydrophobic properties; @brantley2001]. 

More research is needed, including: comparing fungal community and function in mulched vs. unmulched planting beds, the influence of different mulches on fungal colonization and community composition, and mulch as a fungal dispersion vector.

**Clean-up activities:** Landscaping clean-up activities typically include mowing, pruning, and removing plant material (snags, branches, logs, twigs, leaves, grass clippings). These activities could potentially impact fungal communities and function as they impact living plants and include removal of food sources for fungal decomposers. However, research about these activities is uneven and shows mixed results. 

**Mowing:** In tallgrass prairies, researchers found no significant effects of mowing on mycorrhizal root colonization or hyphae development [@eom1999]. However, in subalpine grasslands cessation of mowing led to decreased mycorrhiza densities and increased leaf endophyte infection levels, suggesting an interaction between mowing and fungal functional diversity [@binet2013]. 

**Pruning:** Experiments using pruning to simulate herbivory found that pruning generally can reduce mycorrhizal colonization and alters mycorrhizal fungal community composition [@gehring2002; though see @barto2010]. Further research found that the impacts of pruning on both fungi and plant communities were influenced by the species of arbuscular mycorrhizal fungi (AMF) present [@klironomos2004] and pruning altered mycorrhizal community composition [@ijdo2010; @gehring2002]. These results suggest that changes in fungal community function depend on species present, ecosystem, and host plant species. The impacts of pruning in urban ecosystems has not yet been studied in detail.

**Coarse woody debris removal:** Both fine and coarse woody debris are important for fungal diversity, and both should be allowed to decay naturally whenever possible [@heilmannclausen2003; @heilmannclausen2004; @norden2004; @dove2015]. Rare and endangered fungal species are highly specialized in their resource use and are generally more abundant with increasing amounts of dead wood [connectivity is also important; @norden2013]. In forested systems, fungal species richness generally has been associated with increased dead tree and downed log recruitment [@dove2015].

**Twig, leaf and grass clipping removal:** There is little research examining how fungal communities are impacted by removal of the leaf litter layer. It seems likely that removal of the leaf litter layer removes a food source for decomposer fungi and would impact the fungal community composition and function, however this remains an area for future research [@jumpponen2010].




## Methods {#CH4-methods}

I collected two complimentary fungal datasets: a next-generation sequencing (NGS) dataset based on soil samples and a collection of physical mushroom samples. Community information from the NGS dataset consists of counts of sequence variants (SVs, sometimes called amplicon sequence variants), each representing a unique sequenced gene region [@callahan2017]. Community information in the physical mushroom samples consists of presence/absence data agglomerated at the genus level for each site. 

The NGS sequence variant dataset is the primary dataset used for this analysis, however the two datasets are complimentary. The NGS dataset captures the majority of detectible fungal genetic material present in the soil sample, but cannot distinguish between SVs which are growing and contributing to the ecology on site and SVs which are present only as spores or are biologically inactive [@peay2014]. Sampling above-ground mushrooms can only detect fungal species that are actively fruiting and readily discernable, which misses those that fail to reproduce, reproduce underground, have extremely cryptic fruiting bodies, or are microscopic (e.g. yeasts). This can bias observed community composition as some types of fungi including parasitic and endophytic fungi rarely produce mushrooms [@rodriguez2009]. Strengthening NGS data analysis with physical samples helps to offset the deficiencies of each approach. Though I expect that different communities will be observed using each method, multiple sampling approaches should be more effective than either alone [@hart2015].

This section provides a brief overview of the study area, data collection methods for the soil-derived NGS and mushroom datasets, data collection for independent variables, and data analysis methods.

### Study area

Redmond (2017 pop'n 64,000) and Bellevue (pop'n 144,000) are located east of Seattle in King County, Washington [@Census2017]. Both cities share a similar ecological history, a similar disturbance timeline for logging and agriculture, and both have grown considerably since the opening of the Evergreen Point Floating Bridge (SR 520) in 1963. The adjacent cities are also at similar elevations (<500ft) and experience the same climate and weather.

The sampling frame of office developments was limited to Redmond and Bellevue north of I-90. Developments within Bellevue’s central business district (CBD) were excluded as these high-rise buildings differ significantly from the low-rise office developments outside the CBD.

Study sites were selected using random stratified sampling across five vegetation strata. To avoid confounding factors, I restricted the sampling pool to sites in the 30^th^ to 85^th^ percentile for size and the 15^th^ to 85^th^ percentile for surrounding impervious surfaces.

I sent three rounds of mailings to the property owner or manager on file in the King County Assessor's database (published October 9th, 2014) requesting access to their property [@taxDB]. Of the 46 requests sent, 20 were accepted, 6 were rejected, and 20 received no response or were not deliverable. More information on the site selection can be found in [Chapter 2: Vegetation](#CH2-methods).

My final sample included between 3 and 5 sites from each of the five vegetation strata. The study sites ranged from `r round(min(coll.sample.data$acres), 3)` acres to `r round(max(coll.sample.data$acres), 3)` acres in size. `r length(which(coll.sample.data$DistrictName == "BELLEVUE")) ` sites were located in Bellevue and `r length(which(coll.sample.data$DistrictName == "REDMOND")) + 1` of the study sites were located in Redmond. One of the original Redmond sites was later excluded due to property access conflicts.



### Soil sample collection for nutrient and rRNA analysis 

#### Locating and collecting soil samples {#CH4-methods-soil-sample-collection -}

Soil sampling was carried out at 19 sites---17 office development sites[^17sites] and two adjacent forests---between November 20-22, 2015. Within each office development, I stratified based on ground cover type as urban ground cover results from and leads to different management regimes and ground cover is known to influence fungal communities [Ammirati, personal communication; @trinder2009]. The ground cover types sampled were grass, mulch, and unmanaged areas. Between one and three ground cover types were sampled at each site, depending on which were present at the site. Three samples were collected from each of the two forests. In total, 40 soil samples were collected. Each sample was a composite sample comprising five randomly placed subsamples of the 0-5cm layer of soil. I used subsampling to try to capture more of the heterogeneity of fungal distribution.

To create ground cover types for stratification at each of the 17 office development sites, I digitized ground cover data collected in May and June 2015 in QGIS 2.08 [@QGIS]. I categorized each polygon into four categories: 


* grass; 
* mulch, including recently or previously mulched areas; 
* 'unmanaged' areas with groundcover dominated by native tree litter; and
* inaccessible unsurveyed areas covered with dense landscaping shrubs (e.g. hedges), ivy, water, or hard-packed dirt paths.


Ten survey points were generated for each ground cover type using ArcGIS 10.3 'Create Random Points' tool with a minimum distance between points of 1 ft [@ArcGIS]. Each point was buffered to create a 1 m sampling circle. The ten survey points were labeled 1-10; 1-5 were the 'default' sample points, with 6-10 being used in order as replacement points in case field conditions precluded sampling at one of the first five points. Backup survey points were necessary in a number of cases, including changes to landscaping, misclassification of ground cover (e.g. hard packed dirt present instead of mulch), and presence of soil where the probe could not be inserted to a 5cm depth, which is relatively common in heavily modified urban soils.

For the two forests surveyed, twenty survey points were generated, also using ArcGis 10.3 'Create Random Points' tool with a minimum distance between points of 1 ft. Points 1-15 were grouped into three samples based on proximity to facilitate sampling in dense undergrowth. Points 16-20 were designated back up points. When one of the sampling points was inaccessible, the closest back up points was used instead.

Soil samples were extracted using an AMS 401.03 Soil Recovery Probe (7/8 in x 33in). To collect the sample, the surface layer of mulch, litter, and grass were removed before probe insertion. The probe was firmly pushed into the soil and carefully extracted to avoid sample loss. After extraction, the 0-5cm soil layer was removed from the probe and placed in an 8mil plastic bag. Between samples, but not subsamples, the probe was cleaned using a nylon test tube brush and denatured alcohol. All soil samples for an individual site were placed in a cooler with dry ice at the conclusion of sampling for that site, approximately 30 minutes after the first sample was collected. Soils remained on dry ice until I finished sampling and began soil processing.


[^17sites]: Of the twenty sites in my study, 17 agreed to soil sampling. Three could not be contacted before sampling took place and were excluded from sampling.


#### Soil nutrient analysis {-}

On November 23, all soil samples were thawed, carefully homogenized, and divided in half. One half of each sample was dried for eight days to prepare for CNH, P, and pH testing; the other half was wet processed for rRNA extraction (see \@ref(soil-dna-processing)). 

Once dry, clumps in each sample were physically broken up, the sample passed through a 2 mm sieve, and placed in a fresh plastic zip top bag. Samples were again carefully homogenized and a 3g sub sample extracted. This sub sample was ground in a mortar and pestle and placed in a separate zip top bag for use in Phosphorous testing. All equipment was wiped down between samples with a dry paper towel and a fresh piece of paper was used in place of a sieve pan for each sample. Both bags for each site were labeled 1-40 in no particular order and submitted to the University of Washington School of Environmental and Forest Sciences Analytical Services lab for testing. 

**Carbon, Nitrogen, Hydrogen (CNH):** Total Carbon, Nitrogen, and Hydrogen concentrations were measured using dry combustion (Method 29-2.2, Nelson and Sommers in Page et al. 1982). The Analytical Services lab used a 2400 series CHNS/O elemental analyzer[^CNH]. Based on the Pregl-Dumas method, the analyzer combusts samples in a pure oxygen environment and measures the resulting gases (Perk Elmer Co.).

**Phosphorous (P):** Phosphorous concentration was measured using an inductively coupled plasma-atomic emission spectrometry following the United States Department of Ecology Method 200.7 [@EPAm2007]

**pH:** pH was measured using a HM Digital model PH-80[^pH] pH meter with a glass (indicating) electrode and a calomel (reference) electrode. A standard weight of soil was mixed with water and the pH of this solution was measured following proper standardization with a solution of known pH [Method 12‐2; @mclean1982]. 


[^pH]: See http://www.tdsmeter.com/products/ph80.html.

[^CNH]: See http://www.perkinelmer.com/product/2400-chns-o-series-ii-system-100v-n2410650.


#### Next-generation sequencing  {#NGS -}

I wet sieved the half of the soil sample destined for NGS rRNA processing on November 23. To reduce cross contamination, sieves were thoroughly rinsed and dried between samples, and a fresh piece of paper was used in place of a sieve pan for each sample. All 40 samples were refrozen simultaneously once processing was complete (total time at room temperature for all samples was 7.5 hours). The samples remained frozen at University of Washington in a -20 $^\circ$C freezer[^freezing] until January when they were shipped overnight with dry ice to the Fierer Lab at University of Colorado at Boulder.

[^freezing]: Freezing soil samples at -80 $^\circ$C to -20 $^\circ$C does not affect DNA quality compared with fresh samples [@hart2015].

The Fierer lab extracted DNA from each soil sample using the MoBio PowerSoil DNA[^mobiodoc] isolation kit (MoBio Laboratories, Inc., Carlsbad, CA, USA). Using ITS1-F/ITS2 barcoded primers[^primers], the first internal transcribed spacer (ITS1[^ITS1]) region of the rRNA operon was amplified in triplicate[^Crowther] and sequenced [@crowther2014]. The lab used the PicoGreen dsDNA assay[^pico] to quantify PCR products from all samples before pooling them in equimolar concentrations. The UltraClean PCR Clean-up Kit[^ultra] was used to clean and concentrate amplicons (MoBio Laboratories, Inc., Carlsbad, CA, USA). An Illumina MiSeq instrument using the paired-end v2 300 cycle MiSeq kit[^v2kit] was used to sequence the samples at the University of Colorado Next Generation Sequencing Facility (Illumina, Inc., San Diego, CA, USA). This summary is based on Fierer lab methods described in @barberan2015, @crowther2014, and @rojas2016.

While both forward and reverse reads were obtained from this method, I used only the forward reads for this analysis due to quality issues in the reverse reads causing difficulty pairing the forward and reverse reads. I demultiplexed the forward-read sequences using a Python script[^fiererscript] written by the Fierer lab [@leff2017]. I then used the reverse-complimented ITS1-F and ITS2 sequences and the BBDuk Java script from the BBMap package[^bbmap] to remove any barcoded primers from the demultiplexed sequences with ktrim = r [@bushnell2017]. 


[^fiererscript]: Custom script available at: https://github.com/leffj/helper-code-for-uparse.

[^bbmap]: Online documentation available online at http://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbmap-guide/ and code available at https://sourceforge.net/projects/bbmap/.


I trimmed the first 10 base pairs of each read based on FastQC analysis (Babraham Bioinformatics 2016) and filtered the forward reads using dada2[^dada2] [@callahan2016]. Dereplicated reads were assigned to sequence variants based on error rates learned from the dataset, then used to construct a table with sequence counts for each sample. Note that dada2's approach using modeled error rates allows the creation of sequence variants at very high resolution (to single nucleotide differences). Traditional methods use clustering of sequencing reads at ~97% similarity to produce operational taxonomic units (OTUs). Both the increased resolution and the inherent biological meaning of sequence variants make dada2 and sequence variant pipelines a more advantageous choice than OTU pipelines [@callahan2016].

After removing chimeras[^chimera], the sequence variants were compared with the January 2017 UNITE[^unite] database using a native implementation of the RDP's naïve Bayesian classifier [@koljalg2013; wang2007; @callahan2016]. Accurate taxonomic matching in UNITE is fundamental to trophic assignment and accurate interpretation of results. 
 

[^ITS1]: Many, perhaps all, primers have significant drawbacks. Some have fundamental mismatches with certain taxa, and therefore do not detect them reliably; others detect a region that is 'hyper-variable' in some groups, potentially dividing one species into multiple [@tedersoo2015; @hart2015]. These limitations apply equally across all samples, so while absolute diversity is impacted, between-sample comparisons for my study should not be impacted. 

[^Crowther]: Amplification details from Crowther: "All DNA samples were amplified in triplicate in PCR reactions containing 13 μl water, 10 μl 5 Prime Hot Master Mix, 0.5 μl each of the forward and reverse primers (10 μM final concentration), and 1.0 μl genomic DNA. Reactions were held at 94 $^\circ$C for 3 min, with amplification proceeding for 35 cycles at 94 $^\circ$C for 45 s, 50 $^\circ$C for 60 s, and 72 $^\circ$C for 90 s; a final extension of 10 min at 72 $^\circ$C" [@crowther2014].

[^mobiodoc]: Detailed protocol information found in: https://www.cuny.edu/research/sr/undergrad-research/for-faculty/AREM/PowerSoil_DNA_Isolation_Kit_Instruction_Manual.pdf or here: https://www.qiagen.com/us/resources/download.aspx?id=5c00f8e4-c9f5-4544-94fa-653a5b2a6373&lang=en.

[^primers]: Good diagram outlining ITS primers found here: https://unite.ut.ee/primers.php

[^pico]: Information on PicoGreen: https://tools.thermofisher.com/content/sfs/manuals/mp07581.pdf .

[^ultra]: Detailed protocol information: https://mobio.com/media/wysiwyg/pdfs/protocols/12500.pdf.

[^v2kit]: Information on the MiSeq Reagent Kit v2: https://support.illumina.com/content/dam/illumina-support/documents/myillumina/cebf8b82-b1d9-4384-a64b-002db4193cbe/miseqreagentkit_v2_reagentprepguide_15034097_b.pdf and the MiSeq system: https://www.illumina.com/content/dam/illumina-marketing/documents/products/datasheets/datasheet_miseq.pdf.

[^dada2]: dada2 is an R package that models and corrects Illumina-sequenced amplicon errors and defines sample sequences more accurately than other methods [@callahan2016]. Code and documentation are available online at  https://benjjneb.github.io/dada2/index.html.

[^unite]: UNITE database available online at https://unite.ut.ee/ . Past versions are available for download https://unite.ut.ee/repository.php . 

[^chimera]: Chimeras are hybrid DNA sequences with DNA from more than one organism created during the amplification etc. process [@hart2015].


#### Sequence variant fungal guild assignment {-}

FUNGuild is an annotation tool that joins species tables with fungal guild information based on distributed expert annotations. In other words, mycologists and published papers provide information about different fungal species (e.g. trophic mode, guild assignments) into a database. Although these are traditionally operational taxonomic unit (OTU) tables, here a SV table was used. The Python based tool is fed a species table with taxonomy attached, parses the taxonomy information, matches the taxonomy with the matching database entry (if present), and outputs the annotated taxa table with fungal guild information attached[^funguild] [@nguyen2016]. 

Trophic assignments were based on genus assigned by UNITE for the NGS data or identified in the collections data. We had the greatest level of certainty in this level of taxonomic refinement, as some sequence variants were assigned genus species pairs not known to exist in the United States (e.g. European relatives) and some physical collections could not be identified beyond genus. We excluded sequence variants and collections data that were not identified to genus from trophic analysis as higher taxonomic levels in fungi exhibit significant trophic variation.

I used the online version of the FUNGuild bioinformatic tool available at http://funguild.org to annotate my sequence variant table with fungal guild information. All taxa were assumed to be consistent in their trophic guild; that is, a taxa identified as saprotrophic in forests on hardwood would also be saprotrophic in urban areas. As not all genus found in my datasets are represented in FUNGuild, I supplemented FUNGuild with Google Scholar and other scientific databases to fill in the missing trophic information. This list was corrected and added to by Dr. Joseph Ammirati and myself. 


[^funguild]: Code to run locally is available on Github: https://github.com/thebateslab/FUNGuild.


### Fungal fruiting body data collection 

#### Sites surveyed {#CH4-collection-sites-surveyed -}

I collected fungal fruiting bodies (mushrooms) from 19 of the 20 sites in the larger study. The remaining site was sampled once before access became restricted due to staff turnover.

I carried out site visits over two field seasons between the onset of fall rain and the first major freeze: in 2014 between 7 October and 28 November, and in 2015 between 28 September and 25 November. In the Puget Sound region, October and November are wet fall months when many fungal species fruit [Ammirati, personal communication; @halme2012]. The first major freeze of the year, which generally occurs in late November, destroys the cell structure of most soft-bodied fruiting bodies. Few perennial species fruit after this freeze.

Each site was sampled multiple times to accommodate species specific differences in timing of fruiting [@halme2012; @lodge2004]. In 2014, sampling was limited to 14 of the 19 sites due to private property access negotiations. These sites were visited 1-3 times at 1-2 week intervals over the season. In 2015, all 19 sites were visited 2-3 times at approximately 3 week intervals throughout the sampling season. Between the two years, all sites were sampled 3-5 times.

The number of visits per site impacts overall detectability at each site. Estimates of species group detectability during peak fall fruiting season range from 0.96 for perennial polypores to 0.54 for agarics. This suggests that 2-3 surveys is sufficient if average detectability is greater than 0.5 [@halme2012]. The variable effort between sites (sites were surveyed 2-3 times in 2015) will influence species richness and must be controlled for in analysis.

Additionally, both groundcover and species impact fungal detectability. Groundcover skews the types of fungal species or species groups detected on site. Grass and bare mulch groundcovers rarely impede detection, while dense ivy, shrubs, and heavy leaf litter all reduce the probability of detection. Fungal groups that are frequently associated with dense ivy or shrubs therefore are less likely to be detected. This should impact all sites equally, so while some fungal groups will be under-detected between site comparisons are not impacted.

Unless otherwise noted, only the 2015 data is used in analyses. The uneven sampling schedule, constrained as it was by private property access, presents some challenges. Not all fungi fruit every year, and many do not fruit in successive years [@lodge2004]. While at each visit I conducted a complete census, the fact that five sites were visited only in the second year means that fungi fruiting only in the first year would be missed. As a result, this dataset contains one year of data (2015) with supplemental data from 2014 that should be used with caution. 

This short, though intensive, study period may not be able to capture all species richness at a site. Some guidance suggests that 5-10 years of data is necessary [@lodge2004]. However, others suggest that surveying repeatedly in the same year is more important for detecting species richness than conducting surveys in multiple years [@halme2012]. The short period of the study also minimizes confounding signals from community succession and species turnover.


#### Field Methodology {-}

On each site visit, I carried out a complete site macrofungi census. I methodically searched all accessible pervious surfaces, including all planting beds and lawns, under trees, and on and around deadwood. When I detected a mushroom, I photographed it in situ, collected it, assigned it a unique ID, documented the precise location where it was found, and stored it in a plastic tackle box. Mushrooms in poor condition (rotten, waterlogged, or dead) were not collected, but were photographed and given a unique ID.

Collections were brought back to the lab and stored at 0-5 $^\circ$C, then identified to genus or species by Dr. Joseph Ammirati and preserved via dehydration for future study. At completion of the study, samples will be housed at the University of Washington Herbarium with the exception of four samples identified as _Mycetinis scorodonius f. diminutivus_ [@petersen2017] housed at the University of Tennessee.



### Community data handling and transformations

__NGS dataset:__ I did not aggregate the NGS dataset prior to analysis. Analyses were performed with sequence variants as the biological unit of analysis. For guild-specific analyses (saprotroph, pathotroph, symbiotroph), all sequence variants identified to genus or species were included in analysis without aggregation. I used this approach instead of aggregating by genus as UNITE matches may be missing or unreliable, thus making aggregation unreliable [@koljalg2013]. Aggregation can fail to combine sequence variants without matching UNITE taxonomic information, and generally results either in a dataset with mixed aggregation levels (genus and sequence variants without taxonomic information) or the loss of significant portions of the dataset to achieve uniform aggregation. 

The NGS dataset consists of counts of sequence variants for each soil sample. Generally, NGS datasets are overdispersed and have some unique data properties. Due to differential library read sizes between samples (number of rRNA reads per sample), sample variances are not equal (heteroscedasticity). Further, due to the differential amplification of rRNA sequences between species, you can compare a species between sites but not different species within one site[^dm] [@nguyen2014]. 

The common response to these statistical difficulties is to rarefy NGS (usually as OTU instead of SeqVars) data. Rarefying data involves choosing a 'minimum' acceptable library size (total DNA sequence reads for a sample), discarding any sample where the library size is less than this minimum, and for the remaining samples randomly subsampling without replacement such that all samples are equal in size to the minimum library size [e.g. see @rojas2016; @barberan2015; @mcmurdie2014]. Rarefaction is statistically inefficient (loss of statistical power, increasing Type-II error) and statistically inadmissible (it requires the researcher to omit available valid data according to an arbitrary library size minimum, and adds a random step adding artificial uncertainty). Rarefaction also does not adequately address the overdispersion found in NGS data (increasing Type-I error), can result in a high rate of false positives, and 'solves' heteroscedasticity by equalizing variances based on the largest (worst) value [@mcmurdie2014]. Overall, rarefaction is likely to result in a dataset that does not represent the original communities [@hart2015].

I removed highly rare sequence variants (global n $\leqslant$ 10) from my dataset. Rare sequences have low global read depth---that is, they are detected once or only a few times across the entire dataset. For NGS derived datasets, rare sequences may include PCR/sequencing artefacts or true species detections. However, unless a study is determining precise biodiversity estimates, discarding rare sequences is unlikely to alter the results of downstream community analysis [@brown2015].

In contrast with most other work on the microbiome, I did not rarefy my NGS dataset to an even library size prior to analysis. Instead, prior to PERMANOVA, NMDS, and TITAN analysis I applied the square root transformation to the data to downweight the importance of the most abundant SeqVars [@nguyen2014]. For univariate analysis, I calculated effective number of species [effective species richness; defined as exp(Shannon entropy); see @jost2006] for each sample as the response variable. 

__Collections dataset:__ Collections were aggregated to genus, as this was judged the most accurate resolution for identification. Unlike the NGS dataset where sequence variants have biological meaning and are separated based on true genetic differences, the identification to species of physical samples relies on high quality collected samples. Additionally, since it is not possible to determine the number of distinct organisms of each fungal species are present at each site, collections data is presence absence data. I did not use any rare species filtering for my collections based dataset, and no further transformations were applied prior to multivariate analysis. As with the NGS data, I calculated effective number of species [defined as exp(Shannon entropy); see @jost2006] for the response variable in univariate analysis. 


[^dm]: Note that the Bray-Curtis dissimilarity index is calculated in such a way that the between species/site limitations are not an issue. The following equation is used to calculate Bray-Curtis dissimilarity: $Bcij = 1- 2Cij/(Si + Sj)$ C_ij_ is the sum of the lesser values for only those species found at both sites. S_i_ and S_j_ are the total number of specimens (reads) at both sites. The numerator compares sites within one species, and the denominator is summed library read size, both of which are allowable.


### Independent Variables

#### Detection variables {-}

The number of visits per site, ground cover, and recent weather all impact detection for the collections dataset. To control for these factors, I recorded the number of times each site was visited in 2015 and calculated the proportion of low detection ground cover (dense vegetation and ivy) to total pervious cover. I also calculated total rainfall over the week prior to each sampling event and the median average daily temperature over the 7 days prior to site visitation. A 7 day period was chosen as biologically relevant to fungal fruiting. Weather was not compared between sites per se but rather between the site's sampling events in order to determine if the recent weather conditions when sampling occurred might account for differences in observed mushroom communities. Weather data was obtained from AgWeatherNet, a service of Washington State University. Data was obtained from the Woodinville (21 Acres) sensor located in Woodinville, King County, Washington 98072. The monitoring stations use Campbell Scientific CR-1000 data loggers and sensors from different manufacturers [@agweathernet]. 


#### Vegetation data {-}

__Tree Community:__ At each site, individual trees > 5" DBH were identified to genus or species where possible [@symonds1958tree; @dirr2009manual; @dirr1997hardy; @cope2001muenscher; @viertel1970; @sibley2009; @city-trees; @krussmann1985; @sunset1995]. Unknown species were given an unique identifier for multivariate community analysis. Flexible beta clustering and indicator species analysis were used to identify tree community types. Other tree stand variables, including Douglas fir height as a proxy for tree age, stands predating development via historical records, an native conifer density were calculated as in [Chapter 2 Methods](@CH2-methods).

__Shrub Community:__ At each site, all members of the shrub community were identified to genus or species  [@sunset1995; @krussmann1985; @symonds1963shrub; @sunset1995; @dirr1997hardy; @dirr2009manual]. Unidentified species were given a unique identifier code (e.g. "Unknown A") for multivariate community analysis. Flexible beta clustering and indicator species analysis were used to identify shrub community types as in [Chapter 2 Methods](@CH2-methods).

__Ground Cover:__ Ground cover types were recorded on paper maps, then hand digitized in QGIS to calculate the area of each ground cover type [@QGIS] as in [Chapter 2 Methods](@CH2-methods).

#### Maintenance data {-}

I interviewed each site's landscaping service using the 'elite' interview technique and supplemented these interviews with observational data from site visits [@dexter1970; @harvey2011; @tansey2007; @aberbach2002]. I then assigned a value (Yes, No,  Unknown) to variables based on performance of the following maintenance activities at each site: cleanup (leaf blowing, mowing, etc.), herbicide application, fertilizer application, insecticide application, fungicide application, summer irrigation, much application, and mushroom removal. I removed "Insecticide" and "Fungicide" application from further consideration as there were no "Yes" responses. 

I also censused each site for logs, snags and stumps. Logs needed to be more than 6" in diameter for inclusion. Snags needed to be more than 6" in diameter with the majority more than 6' tall; deadwood not meeting this height requirement was considered a stump. Shrub stumps were not counted towards the total. Multi-trunk stumps were counted only once. Stumps where the tree had grown from a co-dominant trunk were counted. 

#### Parcel and Neighborhood Variables {-}

Parcel variables including site area, year built, building quality, and town, and neighborhood variables including % impervious surfaces within 500 m, % tall tree vegetation cover within 500 m, % short and medium vegetation cover within 500 m, number of major intersections within 500 m, median income, and % foreign-born were all calculated as described in [Chapter 2 Methods](#CH2-methods) and [Chapter 3 Methods](#CH3-methods) [@taxDB; @QGIS; @ACS5yr; @KCDB; @xian2011; @homer2015; @roadsdata; @RedLiDAR; @KingCoLiDAR].



```{r CH4-covariate-definitions, out.width='70%'}
	
	covariate.definitions <- tibble(Metric = c("1.", 
	                                           "**Number of Visits per Site (Collections)**",
	                                           "**Ground Cover Type (NGS)**",
	                                           "**Detectible Ground Cover Proportion (Collections)**",
	                                           "**Median Average Air Temperature (F; Collections)**",
	                                           "**Median Average Soil Temperature @ 8in (F; Collections)**",
	                                           "**Median Total Precipitation (inches; Collections)**",
	
	                                           "2.",
	                                           "**Vegetation Class**",
	                                           "**Tree Cluster Group**",
	                                           "**Shrub Cluster Group**",
	                                           
	                                           "3.",
	                                           "**Area**",
	                                           "**Town**",
	                                           "**Building Age (in 2017)**",
	                                           "**Building Quality**",
	
	                                           "4.",
	                                           "**Impervious w/in 500 m (%)**",
	                                           "**Tall Vegetation w/in 500 m (%)**",
	                                           "**Short and Medium Vegetation w/in 500 m (%)**",
	                                           "**Major intersections w/in 500 m (count)**",
	                                           "**Median Income**",
	                                           "**Percent Foreign-Born**",
	                                                                                      
	                                           "5.",
	                                           "**pH**",
	                                           "**C:N**",
	                                           "**Phosphorous (mcg/g)**",
	
	                                           "6.",
	                                           "**Impervious on Site (%)**",
	                                           "**Stands Predate Development**",
										   "**Density of Native Conifers**",
	                                           "**Median Height of Dominant Douglas-fir**",
	                                           "**Native Shrub Effective Species Richness**",
	                                           "**Density of Native Shrubs**",
	                                           "**Cleanup**",
	                                           "**Irrigation**",
	                                           "**Mulch, Herbicide, and/or Fertilizer Application**",
	                                           "**Mushroom Weeding**",
	                                           "**Dead Wood (count)**"
	                                           ),
	
	                                Definition = c("_DETECTION VARIABLES_",
	                                               "The number of times each office development was censused for mushrooms in fall 2015 (collections dataset only).",
	                                               "Ground cover type of sample (forest, grass, mulch, unmanaged; NGS dataset only).",
	                                               "Proportion of ground cover area that has high detectability of mushrooms (grass, mulch) divided by the total pervious area (collections dataset only).",
	                                               "The median across site visits of the average 7-day air temperature (F) for mushroom collection visits. Average temperature over the seven days prior to visit provided by AgWeatherNet (collections dataset only).",
	                                               "The median across site visits of the average 7-day soil temperature (F) at 8 inches depth for mushroom collection visits. Average temperature over the seven days prior to visit provided by AgWeatherNet (collections dataset only).",
	                                               "Median across site visits of the previous 7-day total amount of precipitation in inches for mushroom collection visits. Total precipitation over the seven days prior to visit provided by AgWeatherNet (collections dataset only).",
	
	                                               "_VEGETATION GROUPS_",
	                                               "Vegetation class assigned prior to site selection. Possible values are described in Chapter 2.",
	                                               "Tree cluster determined based on flexible beta cluster analysis of tree density data (see Chapter 2 for more details). The two clusters are Native and Cultivated trees.",
	                                               "Shrub cluster determined based on flexible beta cluster analysis of shrub density data (see Chapter 2 for more details). The two clusters are Native and Cultivated shrubs.",
	                                               
	                                               
	                                               "_SOCIO-ECONOMIC SITE VARIABLES_", 
	                                               "Parcel area, in acres, based on the King County Assessor's database. Where a business park spanned multiple parcels, I summed the area of all parcels.",
	                                               "The town in which the parcel is located.",
	                                               "Age calculated by subtracting the year that the building on the parcel was constructed, based on the King County Assessor's database, from 2017. Where a business park spanned multiple parcels, I first calculated the mean year built.",
	                                               "An assigned 'quality class' for any buildings on the parcel, based on the King County Assessor's database. There are six possible classes (Low Cost, Low/Average, Average, Average/Good, Good, Good/Excellent) represented in this dataset.",
	
	                                               "_LAND COVER AND SOCIO-ECONOMIC NEIGHBORHOOD VARIABLES_", 
	                                               "The percent of impervious surface over a 500 m buffer, measured from the perimeter of the parcel. Based on the National Land Cover Database 2011 Percent Developed Imperviousness dataset updated in 2014.",
	                                               "The percent of tall tree vegetation canopy cover over a 500 m buffer, measured from the perimeter of the parcel. Based on NDVI calculated using the 2015 NAIP data and object height calculated using 2014 and 2013 LiDAR data.",
	                                               "The percent of short and medium vegetation canopy cover over a 500 m buffer, measured from the perimeter of the parcel. Based on NDVI calculated using the 2015 NAIP data and object height calculated using 2014 and 2016 LiDAR data.",
	                                               "The number of major intersections located within a 500 m buffer, measured from the perimeter of the parcel. Based on data from OpenStreetMap.",
	
	                                               "The median income of residents for the block group in which the parcel is located. Based on the American Community Survey 2014 5-year block group.",
	                                               "The percent of residents born outside of the United States for the block group in which the parcel is located, based on the American Community Survey 2014 5-year block group.",
	
	                                               "_SOIL VARIABLES_",
	                                               "For NGS data, pH of soil measured in water. For collections data, average pH area-weighted by ground cover strata area.", 
	                                               "For NGS data, ratio of Carbon to Nitrogen of soil samples based on measurements. For collections data, ratio of Carbon to Nitrogen of sites, based on measurements area-weighted by ground cover strata area.",
	                                               "For NGS data, concentration of Phosphorous found in soil. For collections data, average Phosphorous concentration area-weighted by ground cover strata area.",
	
	
	                                               "_DEVELOPMENT, LANDSCAPE, AND MAINTENANCE VARIABLES_",
	                                               "Percent of the parcel's area that is occupied by impervious surface. Calculated based on in-person site measurements; primarily determined by development action.",
	                                               "Binary development variable indicating presence of a stand of three+ trees nearby one another that predate development. Primarily determined by development action.",
	                                               "Median height (m) of dominant Douglas-fir, based on a sample of 5 trees. This variable is a proxy for the age of the dominant Douglas-fir, and is primarily determined by tree preservation at development.",
	                                               "Combined density of three major conifer species: Douglas-fir, western redcedar, and western hemlock (three species summed abundance / site area in acres). This variable is determined by both development and landscaping actions.",
	                                               "Effective species richness of observed native shrubs, primarily determined by landscaping actions. Effective species richness is calculated as exp(H') and defined as the number of equally-common species required to give a value of an index, here Shannon-Wiener index, following [@jost2006].",
	                                               "Combined density of all native shrub species (# / site area in acres). Primarily determined by landscaping actions.",
	                                               "Binary maintenance variable indicating whether the landscaping crew removes detritus from the site, including lawn clippings, fall leaves, etc.",
	                                               "Binary maintenance variable indicating whether irrigation is used during the summer months (generally July thru October).",
	                                               "Binary maintenance variables (5) indicating whether landscaping crew applies mulch, herbicides, fertilizers, insecticides, or fungicides to a site.",
	                                               "Binary maintenance variable indicating whether the landscaping crew removes above-ground mushrooms when seen.",
	                                               "Total abundance of dead wood present on site, including stumps, logs, and snags. Maintenance variable."
	
	
	
	)
	)
	
	
knitr::kable(covariate.definitions,
             booktabs = TRUE, 
             caption = "Definitions for all independent variables including detection variables, vegetation groups (see [Chapter 2](#CH2-methods)), site scale socio-economic variables, neighborhood land cover and socio-economic variables,site scale soil variables, and site scale development, maintenance, and landscaping variables.")
	
```
	


#### Independent variable transformations

As in [Chapter 2](#CH2-methods), I standardized total conifer count by total site area and transformed year built to age ($\text{2017} - \text{year built}$). 

For the collections dataset, average soil values (pH, P, C, N) were calculated based on the stratified ground cover samples. Area-based weighting was used for P, C, and N concentrations ($\frac{\sum(\text{GC area * value})}{\sum(\text{all GC areas})}$). pH values were first converted to H+ concentration, area-based weighted average calculated, then converted back to pH values ($-log_{10}\frac{\sum(\text{GC area * H+ concentration})}{\sum(\text{all GC areas})}$).


[^phmath]: See http://www.jerrydallal.com/lhsp/logs.htm.


### Data Analysis

Data analysis methods are summarized in Figure \@ref(fig:CH4-methods).

```{r CH4-methods, out.width='90%'}

	knitr::include_graphics("images/fungimethods.png", auto_pdf = TRUE)
    
```


#### Explaining variation in effective species richness {-}

For each dataset, I used total effective species richness and fungal trophic guild effective species richnes as response variables (saprotrophic, symbiotrophic, and pathotrophic). Effective species richness was calculated as exp(Shannon entropy) [@jost2006]. As in [Chapter 2](#CH2-methods), I created families of related independent variables as described in Table \@ref(tab:CH4-covariate-definitions). 

For each measure of effective species richness, I tested each individual independent variable from a family in a univariate model using the adonis2 {vegan} PERMANOVA implementation with the Euclidian distance matrix to assess variation in effective species richness and partition variation [@vegan; @bakker2012]. For each family of variables, I used the Holm-Bonferroni method to control family-wise error rate using p.adjust() {stats} [@guo2009]. Significance was assessed on adjusted p-values at the $\alpha <= 0.05$ level. I calculated AICc for all single variable models and for all possible models containing significant variables using a custom function based on Residual Sums of Squares.

```{r CH4-methods, out.width='90%', fig.cap='Summary of data analysis methods for fungal communities.'}

	knitr::include_graphics("images/fungimethods.png", auto_pdf = TRUE)
    
```


#### Explaining variation in effective species richness {-}

For each dataset, I used total effective species richness and fungal trophic guild effective species richnes as response variables (saprotrophic, symbiotrophic, and pathotrophic). Effective species richness was calculated as exp(Shannon entropy) [@jost2006]. As in [Chapter 2](#CH2-methods), I created families of related independent variables as described in Table \@ref(tab:CH4-covariate-definitions). 

For each measure of effective species richness, I tested each individual independent variable from a family in a simple univariate model using the adonis2 {vegan} PERMANOVA implementation with the Euclidian distance matrix to partition variation [@vegan; @bakker2012]. For each family of variables, I used the Holm-Bonferroni method to control family-wise error rate using p.adjust {stats} [@guo2009]. Significance was assessed on adjusted p-values at the $\alpha <= 0.05$ level. I calculated AICc for all single variable models and for all possible models containing significant variables using a custom function based on Residual Sums of Squares.

$AICc = 2k + n*ln(\frac{RSS}{n}) + \frac{2k(k+1)}{(n-k-1)}$





#### Explaining variation in fungal community structure {-}

I used PERMANOVA to identify variables that explain variation in fungal community composition or trophic community composition and NMDS for visualization. 

**PERMANOVA:** I used the adonis {vegan} implementation of PERMANOVA in R with the Bray-Curtis distance matrix to partition variation and test differences in fungal community composition and trophic communities [@anderson2001, @vegan]. I first tested a set of independent variables to extract those with significant (alpha < .05) pseudo-F statistics. I used the Holm-Bonferroni method to control family-wise error rate using p.adjust() {stats} [@guo2009]. I used ANOVA to test for significant differences in dispersion [ anova(betadisper()) {stats} {vegan}; @vegan].

I created models of all possible combinations of variables with significant pseudo-F values following Holm-Bonferroni correction. I then calculated AICc using a custom function based on Residual Sums of Squares to compare model parsimony and relative support. 
I used PERMANOVA to identify variables that explain variation in fungal or trophic community composition and NMDS for visualization. 

**PERMANOVA:** I used the adonis2 {vegan} PERMANOVA implementation with the Bray-Curtis distance matrix to partition variation and test differences in bird community and foraging bird communities [@anderson2001, @vegan]. As in univariate analysis, I tested each variable within a family using multivariate simple regression. I then used the Holm-Bonferroni method to control error within the family of variables rate using p.adjust() {stats} [@guo2009]. Significance was assessed on adjusted p-values at the $\alpha <= 0.05$ level. I used ANOVA to test for significant differences in categorical group dispersion using anova(betadisper()) {stats} {vegan} [@vegan].

For variables with significant pseudo-F values following correction,I then calculated AICc using a custom function based on Residual Sums of Squares to compare all possible models using this reduced set of variables.


**NMDS:** Non-metric multidimensional scaling (NMDS) is a rank-based ordination technique based on a distance/dissimilarity matrix [@mccune2002]. I performed NMDS using metaMDS {vegan} using the monoMDS engine based on a Bray-Curtis distance matrix and with autotransform set to false as any necessary transformations were applied prior to analysis [@vegan]. I created visualizations of environmental variables significant in PERMANOVA using the ordihull and ordisurf functions {vegan} [@vegan].


**NMDS:** Non-metric multidimensional scaling (NMDS) is a rank-based ordination technique based on a distance/dissimilarity matrix [@mccune2002]. I performed NMDS using metaMDS {vegan} using the monoMDS engine based on a Bray-Curtis distance matrix and with autotransform set to false as any necessary transformations were applied prior to analysis [@vegan]. I created visualizations of environmental variables significant in PERMANOVA using the ordihull and ordisurf functions {vegan} [@vegan].

For additional detail on NMDS and PERMANOVA methods see [Chapter 2 Methods](#CH2-methods).


#### Environmental gradient response {-}

I examined habitat associations and community change along an ecological gradient for independent variables with significant pseudo-F values in PERMANOVA. I also examined differences in habitat associations for ground cover in the NGS dataset and the tree and shrub groups identified in [Chapter 2](#CH2--methods) for both NGS and collections datasets. 

I used the comparison of ground cover (forest, unmanaged, mulch, grass) using correlation indices to examine differences between the forested sites and other ground covers. I focused on developed sites for environmental gradient analyses using TITAN to facilitate the discussion of multiple variables across their respective gradients, as development and land management variables were only measured on the developed sites. 


**Habitat associations using correlation indices:** For categorical independent variables, I used the multipatt {indicspecies} function to analyze fungal species ecological preferences with correlation indices [@decaceres2010; @indicspecies]. This analyzes the association between a fungal taxa (SV for NGS or genus for mushroom data) and a group of sites [@decaceres2010]. Specifically, correlation indices are "used for determining the ecological preferences of species among a set of alternative site groups or site group combinations" [@de2013].

I used both Pearson's phi coefficient of association, which measures the correlation between two binary vectors (for collections presence matrix) and its abundance-based counterpart the point biserial correlation coefficient (for NGS abundance matrix). I used the "r.g" function to calculate both. As with PERMANOVA, I took the square root of the NGS abundance matrix prior to analysis. As the multipatt function uses permutation, I ran the function 100 times and kept only indicator species present in > 50% of the runs. I then took the mean of the indicator statistics generated for each species, and kept those with a mean index value > 0.5. I took this cutoff from @hinkle2003, who suggest that correlation size 0.5-0.7 is a moderate correlation; 0.7-0.9 a high correlation, and 0.9-1.0 a very high correlation. I interpreted variables where few taxa signaled ecological preferences within these boundaries as not important determinants of ecological preference.

**Gradient analysis using TITAN:** For continuous independent variables describing an environmental gradient, I used the titan {TITAN2} R implementation of Threshold Indicator Taxa Analysis [TITAN; @titan2]. TITAN uses binary partitioning by indicator value [building on the IndVal function in @indicspecies] and is a "method for detecting and interpreting individual taxon contributions to patterns of community change along novel environmental gradients" [@baker2013]. TITAN distinguishes taxa that negatively (z-) or positively (z+) respond to the gradient [@baker2010]. Taxon are evaluated on their purity (consistent direction of response), their reliability (consistent magnitude of response), and whether a narrow range exists for change-point estimation [@baker2010; @baker2013]. Bootstrap replicates are used to estimate the width of change-point estimation; taxa with sharp, nonlinear responses have narrow intervals, while taxa with gradual responses will have wider intervals. Synchronous change in many taxa at a given gradient value can be interpreted as evidence for a community threshold, while asynchronous change in taxa may indicate linear or random response dynamics [@baker2010].

This approach differentiates TITAN from other methods of identifying ecological community thresholds that use univariate indicators (e.g. species richness in regression splines/trees) or multivariate dimension-reduction (e.g. distance matrixes in MRT). TITAN evaluates taxa separately, deconstructing the community to allow assessment of synchrony of taxon-specific change points. The approach is particularly relevant for anthropogenic environmental gradients, which are frequently novel or fall outside of conditions experienced in historical (evolutionary) timelines. Additionally, both rare and common taxa contribute to detection of change-points along the gradient, while other multivariate methods often rely more heavily on predominant taxa [@baker2010].

TITAN requires at least three occurrences of a taxa in a dataset [taxa occurs in at least 3 samples; @titan2]. Taxa in NGS and collections datasets were screened to meet this criteria prior to analysis. I used the square root of the NGS abundance matrix, no transformation was applied to the collections matrix. The purity threshold was set to 0.95 and the reliability to 0.90 for NGS data and 0.75 for collections data, and I used 250 permutation and 500 bootstrap replicates respectively. 



## Results {#CH4-results}

### Fungal observations

Next generation sequencing identified a total of `r sum(Fonly_TrackInfo$input) ` reads. Each of the 40 soil samples returned between `r min(Fonly_TrackInfo$input) ` and `r max(Fonly_TrackInfo$input) ` reads (median = `r median(Fonly_TrackInfo$input) ` reads, mean = `r round(mean(Fonly_TrackInfo$input), 1) ` reads, sd = `r round(sd(Fonly_TrackInfo$input), 1) ` reads). Of these raw reads, `r sum(Fonly_TrackInfo$denoised) ` (`r round(100 * sum(Fonly_TrackInfo$denoised) / sum(Fonly_TrackInfo$input), 1)`%) passed filtering and denoising steps and `r sum(Fonly_TrackInfo$nonchimera) ` (`r 100 * sum(Fonly_TrackInfo$nonchimera) / sum(Fonly_TrackInfo$input)`%) passed chimera detection. Per sample, between `r min(Fonly_TrackInfo$pctKept)`% and `r max(Fonly_TrackInfo$pctKept)`% of the raw samples passed filtering, denoising, and chimera detection (median = `r round(median(Fonly_TrackInfo$pctKept),1)`%, mean = `r round(mean(Fonly_TrackInfo$pctKept), 1)`%, sd = `r round(sd(Fonly_TrackInfo$pctKept), 2)`%).

The UNITE database identified `r nrow(dada2.human)` sequence variants, of which `r nrow(trimmed.dada.taxa) ` sequence variants belonged to the kingdom Fungi and were observed more than 10 times globally (`r round(trimming.amt * 100, 1)`%). The most frequent sequence variant was identified by `r max(rowSums(trimmed.dada.table[ , 2:41])) ` reads (trimmed dataset median = `r median(rowSums(trimmed.dada.table[ , 2:41])) `, mean =  `r round(mean(rowSums(trimmed.dada.table[ , 2:41])),1) `, sd = `r round(sd(rowSums(trimmed.dada.table[ , 2:41])),1) `). `r sum(vegan::specnumber(trimmed.dada.table[ , 2:41]) == 1)` sequence variants were found in only one sample while only `r sum(vegan::specnumber(trimmed.dada.table[ , 2:41]) > 35)` were found in more than 35 of the 40 samples (none were found in all samples). Taxonomic names attributed to the most commonly observed sequence variants were _Hymenoscyphus sp._(39), _Solicoccozyma terricola_ (36),  _Ganoderma sp._ (33), _Penicillium jensenii_ (31), and _Coniochaeta mutabilis_ (31). Sequence variant richness of each sample ranges from `r min(vegan::specnumber(trimmed.dada.table[ , 2:41], MARGIN = 2)) ` to `r max(vegan::specnumber(trimmed.dada.table[ , 2:41], MARGIN = 2))` (median = `r median(vegan::specnumber(trimmed.dada.table[ , 2:41], MARGIN = 2)) ` sequence variants, mean = `r mean(vegan::specnumber(trimmed.dada.table[ , 2:41], MARGIN = 2)) `, sd = `r round(sd(vegan::specnumber(trimmed.dada.table[ , 2:41], MARGIN = 2)),2)`).

The total number of sequence variants identified to genus or species were `r sum(trimmed.dada.taxa$min.tax.rank == "Genus")` and `r sum(trimmed.dada.taxa$min.tax.rank == "Species")` respectively, of which `r  length(unique(paste(trimmed.dada.taxa$Genus, trimmed.dada.taxa$Species))) - 1` genus-species pairs were uniquely identified. Taxonomic duplicates are most frequently genus without associated taxa (representing different species without genetic sequences in UNITE), with some representing within-species genetic variation (due to base-pair differences or unidentified sub species variation). Of the remaining sequence variants, `r sum(trimmed.dada.taxa$min.tax.rank == "Family")` were identified to family, `r sum(trimmed.dada.taxa$min.tax.rank == "Order")` were identified to order, `r sum(trimmed.dada.taxa$min.tax.rank == "Class")` were identified to class, `r sum(trimmed.dada.taxa$min.tax.rank == "Phylum")` were identified to phylum, and `r sum(trimmed.dada.taxa$min.tax.rank == "Kingdom")` were identified only to kingdom.

We collected and identified `r ncol(matrify.collct.genus) ` unique genus of fungi in `r nrow(collections.data[ !(stringr::str_detect(collections.data$ID.Number, "NC")) , ]) ` collected mushroom samples from 19 sites. Of these, `r ncol(matrify.collct.genus[colSums(matrify.collct.genus) == 1]) ` genus were found only on one site and `r ncol(matrify.collct.genus[colSums(matrify.collct.genus) == 19]) ` genus were found on all 19 sites (median = `r median(colSums(matrify.collct.genus)) ` sites, mean = `r round(mean(colSums(matrify.collct.genus)),1)` sites, sd = `r  round(sd(colSums(matrify.collct.genus)) , 1) ` sites). The most commonly observed genus were Mycena (19 sites), Inocybe (18), Amanita (15), Gymnopus (15), Psathyrella (15), and Rickenella (14; summary in Appendix 3). Genus richness at each site ranged from `r min(rowSums(matrify.collct.genus))` to `r max(rowSums(matrify.collct.genus))` species (median = `r median(rowSums(matrify.collct.genus))` species, mean = `r round(mean(rowSums(matrify.collct.genus)),1)`, sd = `r round(sd(rowSums(matrify.collct.genus)),1)`). 

We also identified a new forma, _Mycetinis scorodonius_ f. _diminutivus_. This forma is characterized by diminutive basidiomata, possibly due to the paucity of the substrate on which it is found and other urban stress. All four known samples were found in mulched planters at two of the office developments in this study [@petersen2017]. This suggests that additional forma and species may be discovered in urban areas with further careful collection.



### Independent variables

Weather in October of 2015 was warmer and drier than historical records, while November was significantly cooler and wetter than historical records [2009-2018; @agweathernet]. Precipitation follows a bimodal distribution, likely because mushroom sampling occurred outside of stormy periods (either very little in the past 7 days or multiple inches following a storm). 

```{r CH4-weather-summary, fig.cap="Summary statistics for sampling conditions (average temperature and total precipitation calculated for seven days preceding sampling), monthly average temperature and total precipitation for primary sampling months in 2015, and average October and November conditions over the 2009-2018 period. Data from AgWeatherNet, a service of Washington State University."}


if (word == TRUE) {
    kable(pretty.weather.summary) 
}

if (word == FALSE) {
kable(pretty.weather.summary) %>%
    kable_styling(bootstrap_options = "striped", full_width = F)
}


```

Soil nutrient analysis for NGS samples and agglomerated for mushroom collection samples are shown in Tables \@ref(tab:CH4-soil-NGS-summary) and \@ref(tab:CH4-soil-coll-summary) respectively. Urban soils are heterogenic and exhibit variation in all measures of soil nutrients. Soils in this study were all acidic, with fairly low C:N with the exception of mulched ground cover. 


Soil summary statistics from NGS data:

```{r CH4-soil-NGS-summary}

if (word == TRUE) {
    kable(pretty.NGS.soil.summary, caption = "Summary statistics for soil nutrients. Note that pH median, mean, and standard deviation are reported on the pH values themselves, and not calculated based on the H+. The values are similar (<.2 difference) and sufficient for descriptive statistics.")
}

if (word == FALSE) {
kable(pretty.NGS.soil.summary, caption = "Summary statistics for soil nutrients. Note that pH median, mean, and standard deviation are reported on the pH values themselves, and not calculated based on the H+. The values are similar (<.2 difference) and sufficient for descriptive statistics.") %>%
    kable_styling(bootstrap_options = "striped", full_width = F)
}
```


Soil summary statistics from mushroom collections:

```{r CH4-soil-coll-summary}

if (word == TRUE) {
    kable(pretty.coll.soil.summary, caption = "Summary statistics for soil nutrients. Values are area-weighted.")
}

if (word == FALSE) {
kable(pretty.coll.soil.summary, caption = "Summary statistics for soil nutrients. Values are area-weighted.") %>%
    kable_styling(bootstrap_options = "striped", full_width = F)
}


```


Summary statistics for vegetation, maintenance, and landscaping variables can be found in [Chapter 2](#CH2-results).




### Taxonomic comparison between datasets

The two methods of fungal detection---collecting mushrooms and NGS soil analysis---sample different parts of the fungal community. Collecting mushrooms by hand allows for detection of sparsely distributed macrofungi and taxa not yet recognized in the UNITE database while NGS allows detection of yeasts and micro-fungi. On the 17 sites sampled using both methods, `r length(shared.site.shared.genus)` genera were found with both data collection methods. Unique genera were detected both in the collections (`r length(shared.site.collONLY.genus) ` genera) and NGS datasets (`r length(shared.site.NGSONLY.genus)`; see [Appendix III: Fungi](#CH9-fungiappendix)).

Only two unique genera were found on the sites where mushrooms but no soil samples were collected: `r paste(collections.sites.ONLY.genus, collapse = " and ")`. On the two forested sites where soil samples were collected, `r length(forest.site.ONLY.genus) ` completely unique genera were detected (not found in collections or non-forested NGS samples). 


### Explain variation in effective species richness

No variables were significant in explaining variation in effective species richness for either the NGS or collections datasets. No variables were significant in explaining variation in saprotrophic or symbiotrophic taxa effective species richness for either the NGS or collections datasets. Pathotrophic fungi were not well enough represented in the dataset---parasitic fungi were detected on only 15 of 19 mushroom sampling sites and the number of SV reads for pathotrophic fungi is approximately 1/3 that of saprotrophic fungi.)


### Explain variation in community composition: PERMANOVA Results {#CH4-permanova-results}

Table \@ref(fig:CH4-fungi-var-summary) provides a summary of all PERMANOVA results. AICc comparison tables can be found in [Appendix III: Fungi](#CH9-fungiappendix). Green cells are significant variables where the model y ~ ground cover + variable recieves equals support to y ~ ground cover; yellow cells are significant variables where the model y ~ ground cover + variable did not recieve equal support to the more parsimonious model.


```{r CH4-fungi-var-summary, fig.width = 8, fig.cap ='Summary of all PERMANOVA results for both NGS and mushroom collection datasets. Each variable tested with ground cover (NGS) or individually (MC) then p-values adjusted within colored group. Best AICc model and single variable Delta AICc <0.5 with thick outline.'}

knitr::include_graphics( "images/FungiPERMANOVA1.png", auto_pdf = TRUE)

```


#### NGS PERMANOVA Results {#CH4-ngs-permanova-results}

For the NGS dataset, ground cover type was included in all models as soil sampling was stratified based on ground cover type [Methods](#CH4-methods-soil-sample-collection). Ground cover type alone was significant for all NGS dataset analyses except for the analysis of symbiotic fungi without forested sites included. Ground cover explained `r round(NGS.PERMANOVA.cover.gP[1,1] * 100, 2)  `% of variation, with an average variation per ground cover class explained of `r round(NGS.PERMANOVA.cover.gP[1,2] * 100, 2)  `%. In addition to differences in location found with PERMANOVA, beta dispersion tests suggest that there are also significant differences in 'multivariate spread' or dispersion for the different groups. Tukey tests identified unmanaged sites as the primary source of differences in spread and forested sites as a secondary source. Analysis of plotted data suggests that there are significant differences in both spread and location. No interaction effects between ground cover and site or ground cover and other significant variables were observed. Interaction effects between ground cover and site could not be assessed for analyses including developed sites only due to lack of degrees of freedom.

Soil variables, specifically pH and C:N ratio, were consistently significant in analyses run both with and without forested sites for the full sequence variant dataset as well as saprobic, symbiotic, and parasitic subsets. Tree cluster and shrub cluster groups were also frequently significant, though beta dispersion tests identified significant differences in multivariate spread as well as differences in location. The 'Ornamental Deciduous' group was responsible for this difference, particularly when compared with the forest group. Site area, building characteristics, and land cover and socio-economic neighborhood scale variables were not significant, and vegetation class were also not significant with a few exceptions in any of these analyses (Table \@ref(fig:CH4-fungi-var-summary)). 

``` {r CH4-NMDS-placeholder}
# Add NMDS plot to show differences in pH, C:N, etc?
```


For analyses run without the forested sites included, significant land management variables included those describing the tree and shrub community (including stands predating development, median height of Douglas-fir, and tree and shrub cluster groups). For symbiotic sequence variants, native shrub density and effective species richness were also significant. Variables describing maintenance actions were not significant (including detritus cleanup, irrigation, mulching, herbicide and fertilizer application, mushroom weeding). However, even significant variables explained very little variation in community composition. 

Ground cover alone was the most parsimonious model in most cases (lowest AICc of compared models; see [Appendix III: Fungi](#CH-fungiappendix)). though ground cover and a single other significant variable recieved equal support ($\Delta$ AICc < 2). Generally models with ground cover + 2 or more variables recieved less support. However, AICc values should be interpreted with caution, as models of the form y ~ ground cover + non-significant variables had $\Delta$ AICc of approximately 1.5. A more rigorous cutoff for interpreting equal support may be necessary.



#### Collections PERMANOVA Results {#CH4-coll-permanova-results -}

For the collections dataset, none of the detection variables were significant and were therefore not included as model control variables. Site and neighborhood scale scale variables and neighborhood scale land cover variables were not significant. Tree cluster group membership (native/ornamental deciduous) was significant for both the entire collections dataset and saprobic sub-set. Soil variables were generally not significant, though C:N was significant for the saprobic sub-set of the data. No variables were significant for the symbiotrophic sub-set of the data. 

Of the site scale variables describing development and land management actions, % impervious on site and native conifer density were significant. Other land management variables, including maintenance variables (e.g. irrigation, mulching) were not significant. Significant variables explained approximately 7-15% of variation.

Comparisons of the 2015 only data or combined 2014/2015 data found that generally the same variables were significant though with different Pseudo-F and p-values. The only exception was native shrub effective species richness, which was significant in the combined data only. 2015 only data is reported here (see [Methods](#CH4-collection-sites-surveyed)). 

In general, single variable models received the strongest support, with two variable models receiving equivalent support ($\Delta$ AICc < 2) and three variable models receiving less support. Single variable models with % impervious on site, native conifer density, and tree cluster group recieved the most support, particularly for the full mushroom community and the saprobic mushroom community ($\Delta$ AICc < .3 for both datasets). For the pathotrophic sub-set, percent impervious on-site was the only variable significant in PERMANOVA and received the best support (see [Appendix III: Fungi](#CH-fungiappendix)).

As in the NGS dataset, one variable models---including socio-economic and other variables not significant in PERMANOVA tests---received equivalent support to the best model ($\Delta$ AICc ~ 1). However, these models' adjusted p-values were not significant (see [Appendix III: Fungi](#CH-fungiappendix)). 



### Environmental Gradient Responses

Categorical and binary independent variables describing ecological states and continuous independent variables describing ecological gradients were analyzed using correlation indices and TITAN respectively, due to limits in each approach. Slightly different interpretations are therefore necessary. Using correlation indices tells us which fungal taxa exhibit an ecological preference among the set of habitats represented by the categorical variables. For example, a taxa could prefer the habitat of site groups with stands predating development. TITAN detects changes in taxa distributions along an environmental gradient. Where multiple taxa shift distributions rapidly, we can infer the presence of a change point along the ecological gradient. The taxa reported using TITAN are those that consistently increase or decrease along the environmental gradient. While these taxa do not necessarily strictly prefer habitat of one or the other endpoints, they do exhibit a strong preference (seen in drop-off of occurrence frequency and relative abundance) towards one side of the environmental gradient.

Categorical variables including vegetation clusters identified in [Chapter 2](#CH2-results), ground cover (NGS data only), and land management variables identified as significant in the PERMANOVA results (presence of stands predating development) are analyzed using correlation indices below. Subsequently, ecological gradients represented by continuous variables identified as significant in PERMANOVA (% impervious on site, median dominant Douglas-fir height (m), native conifer density, shrub effective species richness, and native shrub density) and soil variables (pH and C:N) are analyzed using TITAN. 

`r dim(matrify.collct.2015.soil[ ,colSums(matrify.collct.2015.soil > 0) >= 3])[2]` genera in the mushroom collection dataset were present on at least 3 sites and met TITAN's inclusion criteria. For land management variables, `r dim(allNGS.t.TITAN.nf)[2] ` SVs in the NGS dataset met TITAN's inclusion criteria, while `r dim(allNGS.t.TITAN)[2] ` SVs met inclusion criteria for soil variables (includes forested sites). 


#### Fungal taxa ecological preference: substrate preference (NGS only) {-}

Sequence variants in the NGS dataset exhibited ecological preferences for the different substrate types (mulch, grass, unmanaged, forest). Almost all of the preferences detected using the point biserial correlation coefficients were for sequence variants associated with 'unmanaged' habitat (Table \@ref(tab:CH4-point-biserial-NGS-substrate)). These areas of unmanaged remnant forest habitat on office developments were covered with dense shrubs, including salal. This result agrees with PERMANOVA analysis results finding 'unmanaged' habitat different in both location and spread (see [PERMANOVA results](#CH4-permanova-results)). The mean statistic for sequence variants and 'unmanaged' habitat is generally medium to high [@hinkle2003].


```{r CH4-point-biserial-NGS-substrate}

temp <- setorder(pretty.dada.substrate.multipatt[pretty.dada.substrate.multipatt$`Mean Statistic` > .65, ], Substrate, -`Mean Statistic`)

knitr::kable(temp[ , c(8, 9, 3:4)], caption = 'NGS dataset: Fungal genus associations with ground cover types where mean statistic > 0.65. All significant 100% of runs. SeqVar_XXX indicates that no information on the sequence variant is available in the UNITE database.')

```

#### Fungal taxa ecological preference: tree and shrub clusters {#CH4-veg-association-preference -}

Fungi in both the collections dataset and NGS dataset exhibited ecological preferences for the Native tree and shrub vegetation clusters identified in [Chapter 2](#CH2-results). Fungi also exhibited a preference for the Forested sites in the NGS dataset (Table \@ref(tab:CH4-point-biserial-collections-treeshrub) and Table \@ref(tab:CH4-point-biserial-NGS-treeshrub)). The clusters describe the distinct vegetation communities that arise from differences in land management---tree preservation and tree and shrub planting. Cluster designations for both trees and shrubs are Cultivated, Native, and Forest/Undeveloped; the first two cluster groups were identified in [Chapter 2](#CH2-results) and the Forest/Undeveloped cluster is used only for the NGS dataset. Full annotated tables with mean statistic > .5 can be found in [Appendix III: Fung](#CH9-fungiappendix).


```{r CH4-point-biserial-NGS-treeshrub}
library(data.table)


temp <- rbind(pretty.dada.treeclust.multipatt[pretty.dada.treeclust.multipatt$`Mean Statistic`>.65, ],
              pretty.dada.shrubclust.multipatt[pretty.dada.shrubclust.multipatt$`Mean Statistic` > .65, ]) %>%
    setorder(`Vegetation Type`, Cluster) 

knitr::kable(temp[ , c(8, 9, 3:4, 11)], caption = 'NGS dataset: Fungal genus associations with vegetation clusters where mean statistic > 0.65. All significant 100% of runs. SeqVar_XXX indicates that no information on the sequence variant is available in the UNITE database.')

```

```{r CH4-point-biserial-collections-treeshrub}

temp <- rbind(pretty.collect.treeclust.multipatt, pretty.collect.shrubclust.multipatt) %>%
    setorder(`Vegetation Type`, Cluster)

<<<<<<< HEAD
#### Landscaping and maintenance ecological preferences (categorical) {-}
=======
knitr::kable(temp[ , c(1, 9, 3:5, 11)], caption = 'Mushroom collections dataset: Fungal genus associations with vegetation clusters where mean statistic > 0.5.')
>>>>>>> f7d56a60fc2e6a9902e9f6f05b692dd896b4018b

```




#### Fungal ecological preference: development variables (categorical) {-}

Multiple fungal taxa preferred sites with stands predating development, including four sequence variants in the NGS dataset  (Table \@ref(tab:CH4-point-biserial-NGS-SPD)) and 6 genus in the collections dataset (Table \@ref(tab:CH4-point-biserial-collection-SPD)). Only one genus in the collections dataset preferred sites without stands predating development, all others were associated with the presence of stands predating development. Correlation statistics for fungal taxa in both NGS and collections data suggest moderate correlation between taxa and stands predating development.

```{r CH4-point-biserial-NGS-SPD}

#temp <- setorder(pretty.NGS.r.g.standspredate.nf, Substrate, `Mean Statistic`)

knitr::kable(pretty.NGS.r.g.standspredate.nf[ , c(11, 9, 4, 6, 5)], caption = 'NGS dataset: Fungal genus associations with ground cover types where mean statistic > 0.5. All significant 100% of runs.')

```


```{r CH4-point-biserial-collection-SPD}

<<<<<<< HEAD
#### Landscaping and maintenance gradient changes in taxa distribution (continuous) {-}

I analyzed six landscaping and maintenance variables identified as significant in PERMANOVA results with TITAN: % impervious on site, dead wood abundance (cube root), median dominant Douglas fir height (m, squared), native conifer density, shrub effective species richness, and native shrub density. Along all gradients, there were more taxa that consistently decreased in the direction of greater urbanization than increased. TITAN generally provided robust results with NGS data. Though collections data generally agrees with the NGS data, consistent limitations of low number of observations and low numbers of pure and reliable taxa in the collections dataset require a more conservative interpretation. 
=======

temp <- setorder(pretty.collect.r.g.standspredate, Gradient, -`Mean Statistic`)

knitr::kable(temp[ , c(1, 9, 3:5)], caption = 'Mushroom collections dataset: Fungal genus associations with ground cover types where mean statistic > 0.5.')



```




#### Fungal ecological preference: development and landscape variables (continuous) {-}

I used TITAN to analyze six variables describing development and landscaping actions, all identified as significant in PERMANOVA results: % impervious on site, median dominant Douglas fir height (m), native conifer density, shrub effective species richness, and native shrub density. Along all gradients, there were more taxa that consistently decreased in the direction of greater urbanization than increased. TITAN generally provided robust results with NGS data. Though collections data generally agrees with the NGS data, consistent limitations of low number of observations and low numbers of pure and reliable taxa in the collections dataset require a more conservative interpretation. 
>>>>>>> f7d56a60fc2e6a9902e9f6f05b692dd896b4018b

For all graphs, pure and reliable (robust) negatively responding taxa are shown on the left axis, and robust positively responding taxa are shown on the right axis (z-/z+ respectively). Change point symbols (filled or empty circles for z-/z+) are plotted at the median of the bootstrap replicates using the median z-score instead of the observed value. The horizontal lines extending from each change point (solid or dashed lines for z-/z+) represent the 5%-95% quantiles based on the bootstrapped change point distribution. The vertical lines represent the 50% quantile or median sum(z) of the change points for the increasing or decreasing community, determined by resampling the observed data. The hashed box represents the 5%-95% quartiles of sum(z) of change points for the increasing and decreasing community. Unless otherwise indicated, these change points corresponding to declining/increasing taxa for the robust taxa. For all graphs, a red vertical line plus pink hash is used for the "more urban" end of the environmental gradient, while a green vertical line plus gray hash is used for the "less urban" end of the environmental gradient. Please see [https://cran.r-project.org/web/packages/TITAN2/vignettes/glades.TITAN.html] for more information.

Underneath each change point graph is a count-based histogram of the surveyed sites over the environmental gradient. These are provided to show where along the environmental gradient there are gaps in surveyed sites. Gaps in sampling the environmental gradient may increase the width of the bootstrapped change point distribution for each taxa (horizontal lines).

**Percent impervious surface on site:** Fewer taxa had positive associations with increasing percent impervious on site in both NGS and collections datasets. A synchronous change in both increasing (z+) and decreasing (z-) taxa was observed at approximately 70% impervious surface on site, including more than 20 decreasing (z-) and 15 increasing (z+) taxa for the NGS dataset. NGS and collections datasets provided very similar results; median filtered sum(z-) was `r NGS.TITAN.imp.nf$sumz.cp["fsumz-", "0.50"] ` for the NGS dataset and `r coll.TITAN.imp$sumz.cp["fsumz-", "0.50"] ` for the collections data; median filtered sum(z+) was `r NGS.TITAN.imp.nf$sumz.cp["fsumz+", "0.50"]  ` for the NGS dataset and unfiltered sum(z+) was `r coll.TITAN.imp$sumz.cp["sumz+", "0.50"] ` for the collections data.

The bootstrap frequency distributions of decreasing (z-) taxa with change points around 70% generally extended towards lower, but not higher, amounts of impervious surface. Overall, there is evidence for a community threshold at approximately 70% impervious surface. Additionally, below 70% impervious surface on site there is a wide, asynchronous distribution of decreasing (z-) taxa change points, suggesting that additional taxa drop out gradually. Relatively few sites were sampled between 60-70% and below 50% impervious surface. Further sampling of office developments with lower % impervious surface on site may reveal additional community thresholds at these levels.

NGS data:

```{r CH4-TITAN-pct-impervious, fig.width=8.5, fig.height=9.5}

mat <- matrix(c(1,2), ncol = 1, nrow = 2)
layout(mat = mat, heights = c(9,2))

plotTaxaKD(NGS.TITAN.imp.nf, ntick = 20, bty = NULL, xmin = 0.49, xmax = .85, z.med = T,
           leg.x = .51, leg.y = 6, 
           xlabel = "% Impervious Surface on Site",
           sub1.labs.function = "pretty.labs",
           sub2.labs.function = "pretty.labs",
           margin = c(1,11,3,8)
           )
    rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.imp.nf$sumz.cp["fsumz+", "0.05"],
         xright = NGS.TITAN.imp.nf$sumz.cp["fsumz+", "0.95"], col = "pink", density = 10)
    abline(v = NGS.TITAN.imp.nf$sumz.cp["fsumz+", "0.50"] , col = "red", lwd = 2.5)
    
    rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.imp.nf$sumz.cp["fsumz-", "0.05"],
         xright = NGS.TITAN.imp.nf$sumz.cp["fsumz-", "0.95"], col = "gray65", density = 10)
    abline(v = NGS.TITAN.imp.nf$sumz.cp["fsumz-", "0.50"] , col = "yellowgreen", lwd = 2.5)

    par(mar = c(3, 7, .5, 8))
hist(NGS.sample.data.nf$impervious.sqft, breaks = 17,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)
```

Mushroom collections data:

```{r CH4-TITAN-pct-impervious-c, fig.width=8.5, fig.height=8}
layout(mat = mat, heights = c(9,2))

plotTaxaKD(
    coll.TITAN.imp,
    all = T,
    leg.x = .5,
    margin = c(1, 10, 3, 8),
    z.med = T,
    fil1 = "black",
    xlabel = "% Impervious Surface on Site",
    xmin = .42,
    xmax = .85
    )
    abline(v = coll.TITAN.imp$sumz.cp["fsumz+", "0.50"] , col = "red", lwd = 2.5)
    abline(v = coll.TITAN.imp$sumz.cp["fsumz-", "0.50"] , col = "yellowgreen", lwd = 2.5)

par(mar = c(3, 9.8, .5, 8))
hist(NGS.sample.data.nf$impervious.sqft, breaks = 19,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)


```


**Median Dominant Douglas-fir Height (m):** More taxa were associated with increasing median dominant Douglas-fir height (m) in both NGS and collections datasets. In the NGS dataset, negative (z-) indicator taxa declined gradually between 5 and 25 m. Median filtered sum(z-) was `r NGS.TITAN.height.nf$sumz.cp["fsumz-", "0.50"] `. Change points for these z- taxa also have relatively wide bootstrap intervals. The asynchrony of change points for z- taxa is not consistent with an ecological community threshold among fungi associated with smaller Douglas-fir. 

In contrast, positive (z+) indicator taxa increased fairly sharply between 27 and 37 m. The median filtered sum(z+) was `r NGS.TITAN.height.nf$sumz.cp["fsumz+", "0.50"] `. The sharp accumulation of taxa with increasing median height and relatively narrow change point bootstrap intervals suggests that there may be a defined ecological community threshold with increasing median Douglas-fir height around 35 m.

Analysis using the collections dataset should be interpreted with caution due to the low number of observations and of pure and reliable taxa (Figure \@ref(fig:CH4-TITAN-median-height-c)). A similar pattern as in the NGS dataset is apparent when including all taxa with significant p-values < 0.05 instead of only pure and reliable taxa. Median filtered sum(z-) was  `r coll.TITAN.height$sumz.cp["fsumz-", "0.50"] ` and median filtered sum(z+) was `r coll.TITAN.height$sumz.cp["fsumz+", "0.50"] ` for the collections data.

NGS data:

``` {r CH4-TITAN-median-height, fig.width=8.5, fig.height=10.5}

mat <- matrix(c(1,2), ncol = 1, nrow = 2)
layout(mat = mat, heights = c(9,2))

plotTaxaKD(NGS.TITAN.height.nf, bty = NULL, z.med = T, xmax = 42,
           leg.x = 0, leg.y = 35,
           xlabel = "Median Dominant Douglas-fir Height (m)",
           sub1.labs.function = "pretty.labs",
           sub2.labs.function = "pretty.labs",
           margin = c(1,9.5,3,10)
           )
    rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.height.nf$sumz.cp["fsumz+", "0.05"],
         xright = NGS.TITAN.height.nf$sumz.cp["fsumz+", "0.95"], col = "gray65", density = 10)
    abline(v = NGS.TITAN.height.nf$sumz.cp["fsumz+", "0.50"] , col = "yellowgreen", lwd = 2.5)
    
    rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.height.nf$sumz.cp["fsumz-", "0.05"],
         xright = NGS.TITAN.height.nf$sumz.cp["fsumz-", "0.95"], col = "pink", density = 10)
    abline(v = NGS.TITAN.height.nf$sumz.cp["fsumz-", "0.50"] , col = "red", lwd = 2.5)

par(mar = c(3, 9.5, .5, 10))
hist(NGS.sample.data.nf$height.m.median, breaks = 17,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)
```

Mushroom collections data:

``` {r CH4-TITAN-median-height-c, fig.width=8.5, fig.height=10.5}
layout(mat = mat, heights = c(9,2))

plotTaxaKD(coll.TITAN.height, all = T, xmin = 0, xmax = 42,
           xlabel = "Median Dominant Douglas-fir Height (m)",
           margin = c(1,5,3,5), z.med = T,
           leg.y = 12, leg.x = 0)
abline(v = coll.TITAN.height$sumz.cp["fsumz+", "0.50"], col = "yellowgreen", lwd = 2)
abline(v = coll.TITAN.height$sumz.cp["fsumz-", "0.50"], col = "red", lwd = 2)

par(mar = c(3, 5, .5, 4.5))
hist(NGS.sample.data.nf$height.m.median, breaks = 19,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)


# z- is fairly synchronous change points, with relatively narrow bootstrap intervals. specialist taxa in younger forests?
# z+ is asynchronous, with a gradual accumulation of taxa with increasing median height. suggests these are generalists in older forests.

```


**Native Conifer Density (trees/acre):** In both NGS and collections datasets, there were more positive (z+) indicator taxa then negative (z-). In the NGS dataset, negative indicator taxa declined sharply between 0-4 native conifer trees/acre, resulting in a median filtered sum(z-) of `r NGS.TITAN.tcnd.nf$sumz.cp["fsumz-", "0.50"] ` trees/acre. The overlap of individual taxa change points provides evidence for an ecological community threshold. Positive (z+) indicator taxa exhibit two sharp inclines, the first between 22-25 trees/acre and the second between 40-50 trees/acre. The median filtered sum(z+) is  `r NGS.TITAN.tcnd.nf$sumz.cp["fsumz+", "0.50"] `. Plots of the sum(z) also support two discrete taxa change points. Distribution of z+ bootstrap replicates is wide, though relatively few observations at higher native conifer densities suggest interpreting these results with caution. 

For the collections dataset, similar patterns in positive (z+) and negative (z-) indicator taxa were apparent when including all taxa with significant p-values < 0.05 instead of only pure and reliable taxa. The low number of observations and pure and reliable taxa suggest that the dataset should be interpreted with caution. Median filtered sum(z-) was `r coll.TITAN.tcnd$sumz.cp["fsumz-", "0.50"] ` and median filtered sum(z+) was `r coll.TITAN.tcnd$sumz.cp["fsumz+", "0.50"] ` for the collections data.

NGS data:

``` {r CH4-TITAN-conifer-density, fig.width=8.5, fig.height=10.5}

mat <- matrix(c(1,2), ncol = 1, nrow = 2)
layout(mat = mat, heights = c(9,2))

plotTaxaKD(NGS.TITAN.tcnd.nf, bty = NULL, z.med = T,
           leg.x = 50, leg.y = 10,
         xlabel = "Native Conifer Density (#/acre)",
           sub1.labs.function = "pretty.labs",
           sub2.labs.function = "pretty.labs",
           margin = c(1,8,3,8)
           )
     rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.tcnd.nf$sumz.cp["fsumz+", "0.05"],
         xright = NGS.TITAN.tcnd.nf$sumz.cp["fsumz+", "0.95"], col = "gray65", density = 10)
    abline(v = NGS.TITAN.tcnd.nf$sumz.cp["fsumz+", "0.50"] , col = "yellowgreen", lwd = 2.5)
    
    rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.tcnd.nf$sumz.cp["fsumz-", "0.05"],
         xright = NGS.TITAN.tcnd.nf$sumz.cp["fsumz-", "0.95"], col = "pink", density = 10)
    abline(v = NGS.TITAN.tcnd.nf$sumz.cp["fsumz-", "0.50"] , col = "red", lwd = 2.5)

par(mar = c(3, 7.8, .5, 7))
hist(NGS.sample.data.nf$tree.conifer.nat.dens, breaks = 17,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)


# plotSumz(NGS.TITAN.tcnd.nf, filter = T, cumfrq = F)

```

Mushroom collections data:

``` {r CH4-TITAN-conifer-density-c, fig.width=8, fig.height=10}
layout(mat = mat, heights = c(9,2))

plotTaxaKD(coll.TITAN.tcnd, all = T, xmin = 0, xmax = 60,
           xlabel = "Native Conifer Density (#/acre)",
           margin = c(1,5,3,5.5), z.med = T,
           leg.y = 12, leg.x = 40)
abline(v = coll.TITAN.tcnd$sumz.cp["fsumz+", "0.50"], col = "yellowgreen", lwd = 2)
abline(v = coll.TITAN.tcnd$sumz.cp["fsumz-", "0.50"], col = "red", lwd = 2)

par(mar = c(3, 5, .5, 3.7))
hist(NGS.sample.data.nf$tree.conifer.nat.dens, breaks = 19,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)


# z+ primary peak at 20, secondary peak at about 40-50, appear to be multiple
# discrete taxa change points; distribution of bootstrap replicates is
# wide--relatively few observations at higher native conifer densities suggest
# interpreting these results with caution.




```


**Shrub Effective Species Richness:** There were more positive (z+) indicator taxa than negative (z-) for both NGS and collections datasets. For the NGS dataset, positive (z+) indicator taxa increased sharply between 3.5-4.0 effective species, with a median filtered sum(z+) of `r NGS.TITAN.sesr.nf$sumz.cp["fsumz+", "0.50"]` effective species. The synchrony of change, which includes over 20 taxa, provides support for an ecological community threshold. Additionally, the width of the distribution of bootstrap replicates were generally narrow, suggesting this threshold involves relatively sensitive taxa. Conversely, negative (z-) indicator taxa dropped out in an approximately linear sequence between 1.5-3.0 effective species, suggesting an asynchronous distribution of negative (z-) taxa change points. The median filtered sum(z-) is `r NGS.TITAN.sesr.nf$sumz.cp["fsumz-", "0.50"] ` effective species. Bootstrap replicates for negative taxa are generally wide, though fewer samples between 1-3 effective species suggest interpreting the results with caution.

For the collections dataset, positive (z+) indicator taxa showed a similar pattern, with evidence of an ecological community threshold between 3.7-4.2 effective species (Figure). The median filtered sum(z+) for this dataset was `r coll.TITAN.sesr$sumz.cp["fsumz+", "0.50"] ` effective species. Negative (z-) indicator taxa were not well defined and an unfiltered median sum(z-) could not be calculated due to too few pure and reliable taxa and observations. 

NGS data:

```{r CH4-TITAN-shrub-esr, fig.width=8.5, fig.height=10.5}

mat <- matrix(c(1,2), ncol = 1, nrow = 2)
layout(mat = mat, heights = c(9,2))

plotTaxaKD(NGS.TITAN.sesr.nf, bty = NULL, z.med = T,
           leg.x = .9, leg.y = 15, xmax = 3.6,
           xlabel = "Shrub Effective Species Richness",
           sub1.labs.function = "pretty.labs",
           sub2.labs.function = "pretty.labs",
           margin = c(1,8,3,8)
           )

    rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.sesr.nf$sumz.cp["fsumz+", "0.05"],
         xright = NGS.TITAN.sesr.nf$sumz.cp["fsumz+", "0.95"], col = "gray65", density = 10)
    abline(v = NGS.TITAN.sesr.nf$sumz.cp["fsumz+", "0.50"] , col = "yellowgreen", lwd = 2.5)
    
    rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.sesr.nf$sumz.cp["fsumz-", "0.05"],
         xright = NGS.TITAN.sesr.nf$sumz.cp["fsumz-", "0.95"], col = "pink", density = 10)
    abline(v = NGS.TITAN.sesr.nf$sumz.cp["fsumz-", "0.50"] , col = "red", lwd = 2.5)

par(mar = c(3, 7.8, .5, 5))
hist(NGS.sample.data.nf$shrub.esr, breaks = 17,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)


# plotSumz(NGS.TITAN.sesr.nf, filter = T, cumfrq = F)
```

Mushroom collections data:

```{r CH4-TITAN-shrub-esr-c, fig.width=8, fig.height=10}
layout(mat = mat, heights = c(9,2))

plotTaxaKD(coll.TITAN.sesr, all = T, xmin = 1, xmax = 4.1,
           xlabel = "Shrub Effective Species Richness",
           margin = c(1,5.1,3,6), z.med = T,
           leg.y = 6, leg.x = 1)
abline(v = coll.TITAN.sesr$sumz.cp["fsumz+", "0.50"], col = "yellowgreen", lwd = 2)
abline(v = coll.TITAN.sesr$sumz.cp["fsumz-", "0.50"], col = "red", lwd = 2)

par(mar = c(3, 4.9, .5, 5))
hist(NGS.sample.data.nf$shrub.esr, breaks = 19,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)


```


**Native Shrub Density:** The NGS, but not the collections dataset, found more positive (z+) indicator taxa than negative (z-) taxa. Most of the detected negative (z-) indicator taxa in the NGS dataset declined in a sharp linear pattern between 0-25 native shrubs/acre, with a median filtered sum(z-) of `r NGS.TITAN.shrubdens.nf$sumz.cp["fsumz-", "0.50"]` shrubs/acre. This relatively confined band of nearly all detected negative (z-) taxa suggest an ecological community threshold. 

Detected positive (z+) indicator taxa exhibit two disjoint clusters of change points in response to increasing native shrub density: the first between 20-45 native shrubs/acre and the second around 100 shrubs per acre, each consisting of over 20 taxa. Median filtered sum(z+) falls in the middle at `r NGS.TITAN.shrubdens.nf$sumz.cp["fsumz+", "0.50"]`, with wide bounds (grey box, Figure). The first cluster of change points matches with the ecological community threshold suggested by negative (z-) indicator taxa. The second suggest an additional community change at higher shrub densities. However, that few sites surveyed had greater than 50 native shrubs/acre limits interpretability both of this second ecological community threshold and the width of the distribution of bootstrap replicates.

Both few detected pure and reliable taxa and the low number of observations limit interpretability of the collections dataset. For the negative (z-) taxa, the collections dataset suggests a similar pattern as the NGS dataset with some evidence of an ecological community threshold at about 20 native shrubs/acre. However, median sum(z-) for the filtered data could not be calculated, for unfiltered the median sum(z-) is `r coll.TITAN.shrubdens$sumz.cp["sumz-", "0.50"]`. Positive taxa appear to increase either in discrete clusters as with the NGS data or linearly with increasing shrub density. Median filtered sum(z+) is `r coll.TITAN.shrubdens$sumz.cp["fsumz+", "0.50"]`.

NGS data:

``` {r CH4-TITAN-shrub-density, fig.width=8, fig.height=10}

mat <- matrix(c(1,2), ncol = 1, nrow = 2)
layout(mat = mat, heights = c(9,2))

plotTaxaKD(NGS.TITAN.shrubdens.nf, bty = NULL, z.med = T,
           leg.x = 90, leg.y = 10,
           xlabel = "Native Shrub Density (#/acre)",
           sub1.labs.function = "pretty.labs",
           sub2.labs.function = "pretty.labs",
           margin = c(1,8,3,8)
           )
     rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.shrubdens.nf$sumz.cp["fsumz+", "0.05"],
         xright = NGS.TITAN.shrubdens.nf$sumz.cp["fsumz+", "0.95"], col = "gray65", density = 10)
    abline(v = NGS.TITAN.shrubdens.nf$sumz.cp["fsumz+", "0.50"] , col = "yellowgreen", lwd = 2.5)
    
    rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.shrubdens.nf$sumz.cp["fsumz-", "0.05"],
         xright = NGS.TITAN.shrubdens.nf$sumz.cp["fsumz-", "0.95"], col = "pink", density = 10)
    abline(v = NGS.TITAN.shrubdens.nf$sumz.cp["fsumz-", "0.50"] , col = "red", lwd = 2.5)

par(mar = c(3, 7.8, .5, 6.4))
hist(NGS.sample.data.nf$shrub.nat.dens, breaks = 17,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)


# plotSumz(NGS.TITAN.shrubdens.nf, filter = T, cumfrq = F)
```

Mushroom collections data:

``` {r CH4-TITAN-shrub-density-c, fig.width=8, fig.height=10}
layout(mat = mat, heights = c(9,2))

plotTaxaKD(coll.TITAN.shrubdens, all = T, xmin = 0, 
           xlabel = "Native Shrub Density (#/acre)",
           margin = c(1,5,3,9), z.med = F,
           leg.y = 6, leg.x = 80)
abline(v = coll.TITAN.shrubdens$sumz.cp["fsumz+", "0.50"], col = "yellowgreen", lwd = 2)
abline(v = coll.TITAN.shrubdens$sumz.cp["fsumz-", "0.50"], col = "red", lwd = 2)

par(mar = c(3, 5, .5, 3.7))
hist(NGS.sample.data.nf$shrub.nat.dens, breaks = 19,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)


# z+: two distinct 
# 
# Few sites measured above 50 native shrubs/acre limits interpretability, particularly of the bootstrap replicates.
# 
# z-: narrow band suggests ecological community threshold.



```


#### Fungal ecological preference: Soil chemistry gradients {-}

Soil chemistry gradient changes is reported for the NGS dataset with forested sites included only, as the reduced number of sites with both collections data and soil data was insufficient for analysis. 

**pH:** Approximately equal numbers of negative (z-) and positive (z+) indicator taxa were identified along the pH gradient. Both categories of indicator taxa experienced sharp changes around pH 5.5. Median filtered sum(z-) was `r NGS.TITAN.pH.f$sumz.cp["fsumz-", "0.50"] `; median filtered sum(z+) was `r NGS.TITAN.pH.f$sumz.cp["fsumz+", "0.50"]  ` for the NGS dataset. Additional taxa fell out (z-) or were added (z+) in an approximately linear sequence both below and above pH 5.5. This synchronous change in taxa is consistent with an ecological community threshold.


``` {r CH4-TITAN-pH, fig.width=8.5, fig.height=10.5}

mat <- matrix(c(1,2), ncol = 1, nrow = 2)
layout(mat = mat, heights = c(9,2))

plotTaxaKD(NGS.TITAN.pH.f, bty = NULL, z.med = F,
           leg.x = 6.2, leg.y = 6, xmin = 4.5,
         xlabel = "pH",
           sub1.labs.function = "pretty.labs",
           sub2.labs.function = "pretty.labs",
           margin = c(1,8,3,8)
           )
     rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.pH.f$sumz.cp["fsumz+", "0.05"],
         xright = NGS.TITAN.pH.f$sumz.cp["fsumz+", "0.95"], col = "gray65", density = 10)
    abline(v = NGS.TITAN.pH.f$sumz.cp["fsumz+", "0.50"] , col = "yellowgreen", lwd = 2.5)
    
    rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.pH.f$sumz.cp["fsumz-", "0.05"],
         xright = NGS.TITAN.pH.f$sumz.cp["fsumz-", "0.95"], col = "pink", density = 10)
    abline(v = NGS.TITAN.pH.f$sumz.cp["fsumz-", "0.50"] , col = "red", lwd = 2.5)

par(mar = c(3, 7.4, .5, 9))
hist(NGS.sample.data$pH, breaks = 17,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)


# plotSumz(NGS.TITAN.pH.f, filter = T, cumfrq = F)


```


**Carbon to Nitrogen Ratio:** Both negative (z-) and positive (z+) indicator taxa have asynchronous changes over the environmental gradient. Median filtered sum(z-) was `r NGS.TITAN.C.N.f$sumz.cp["fsumz-", "0.50"] ` and the median filtered sum(z+) was `r NGS.TITAN.C.N.f$sumz.cp["fsumz+", "0.50"]  `. The widely distributed changes in both indicator taxa together span most of the range of the C:N gradient. This suggests a gradual replacement of taxa tolerant of low C:N values with those tolerant of high C:N values, instead of an ecological community threshold.


``` {r CH4-TITAN-CN, fig.width=8.5, fig.height=10.5}

mat <- matrix(c(1,2), ncol = 1, nrow = 2)
layout(mat = mat, heights = c(9,2))

plotTaxaKD(NGS.TITAN.C.N.f, bty = NULL, z.med = T,
           leg.x = 37, leg.y = 15, 
           xlabel = "C:N",
           sub1.labs.function = "pretty.labs",
           sub2.labs.function = "pretty.labs",
           margin = c(1,8,3,8)
           )
     rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.C.N.f$sumz.cp["fsumz+", "0.05"],
         xright = NGS.TITAN.C.N.f$sumz.cp["fsumz+", "0.95"], col = "gray65", density = 10)
    abline(v = NGS.TITAN.C.N.f$sumz.cp["fsumz+", "0.50"] , col = "yellowgreen", lwd = 2.5)
    
    rect(ybottom = -10, ytop = 100, xleft = NGS.TITAN.C.N.f$sumz.cp["fsumz-", "0.05"],
         xright = NGS.TITAN.C.N.f$sumz.cp["fsumz-", "0.95"], col = "pink", density = 10)
    abline(v = NGS.TITAN.C.N.f$sumz.cp["fsumz-", "0.50"] , col = "red", lwd = 2.5)

par(mar = c(3, 7.4, .5, 4.5))
hist(NGS.sample.data$C.N, breaks = 17,
     main = NULL, xlab = NULL, ylab = NULL, freq = T)


# plotSumz(NGS.TITAN.pH.f, filter = T, cumfrq = F)


```








## Discussion {#CH4-discussion}

Univariate analysis of aggregated measures of the urban fungal community---including species richness and proportion of observed taxa that are saprotrophs---found that broad scale the fungal community are driven largely by stochastic events. However, multivariate analysis found that measures of current development, land management, and soil characteristics, but not neighborhood scale patterns were significant predictors of fungal community composition. Evaluating taxa's responses to environmental gradients found strong responses in some taxa, though fewer than expected. These patterns were observed in both measures of the urban fungal community---next-generation sequencing and mushroom collections.

These results suggest that most taxa's distribution on office developments is stochastic while a subset of taxa respond strongly to soil patterns and development and landscape design actions. Yet unknown is the impact of plot scale land use history, including specific agricultural crops and practices and previous development. The importance of recent land management actions contributing to novel ecosystem process to the distribution of specific taxa and their respective ecosystem function suggests that urban planners, landscape architects, property owners, and land managers should take a precautionary approach in urban ecosystems. 


### Bias in measures of the fungal community

Understanding the limitations of the datasets collected is critical to precisely interpreting results. The two datasets collected as part of this study measured the fungal community in distinct but complimentary ways: next-generation sequencing captures information on ITS-readable fungal genetic information in soil, while collecting mushrooms identifies macrofungi fruiting on site [@hart2015]. 

However, these two common methods of data collection are not yet fully integrated, which limits our ability to compare these datasets and understand taxonomic diversity [@hibbett2016]. Specifically, many sequence variants identified through NGS analysis were not associated with known fungal species/taxa. While many of these unassociated sequence variants may be unknown strains of described species instead of truly new species [@nagy2011], significant work remains to match DNA to taxonomy. Matching sequences to historical type specimens and herbarium collections is an important area of further research, however, it is difficult due to technical challenges with extracting DNA from old specimens. Working with new specimens is significantly easier but requires time and effort to collect and sequence new material [@truong2017]. 

Large gaps in DNA-to-taxonomy databases, including the UNITE database used in this research, mean that large numbers of unknown species are present in my NGS dataset from office developments (See [Appendix III: Fungi](#CH9-fungiappendix)). Two important biases result. First, analyses using the NGS dataset in this study are technically measuring genetic diversity instead of taxonomic or species diversity. Genetic diversity includes any difference in sequence, including genetic variations within species, while taxonomic or species diversity includes species richness, and traditional calculations of alpha and beta diversity. While some studies agglomerate to genus level and drop the 'unknown' SVs/OTUs [e.g. @karpati2011], this eliminates a large portion (>50%) of the dataset based on arbitrary criteria--specifically, whether a sequence can be matched to a taxonomic identification in UNITE or another database. Retaining unknown SVs/OTUs is also problematic and results in disparate aggregation levels where some unidentified sequences may be a genetic variant of an aggregated species, effectively double counting (or more) the species. Further, different rates of study and sequencing of fungal taxa means that certain lineages---including understudied and difficult to culture fungi---are under-represented in UNITE and other databases. Any agglomeration will misrepresent the relative importance of these taxa. Therefore, I believe that agglomerating based on matched taxonomy is methodologically and ecologically unsupportable.

Second, only a subset of sequence variants that should be included in trophic guild analyses are included, as a significant number of SVs could not be identified to genus and therefore could not be assigned to a trophic guild (saprotroph, symbiotroph, pathotroph). Unlike with the agglomeration discussed above, this subsetting of the data cannot be avoided. Trophic guild analyses are therefore biased towards species that have been successfully cultured or collected and sequenced [@hibbett2016]. How biased this subset is cannot be easily assessed--those not included generally don't have enough taxonomic information to make meaningful between-group comparisons.

The mushroom dataset avoids these limitations, which partially helps mitigate bias with the NGS dataset [@hart2015]. However, collections miss yeasts and molds, along with fungal species that rarely fruit [@rodriguez2009]. These are important to ecosystem functioning. Any results from this dataset must be thought of in terms of the community of visually identifiable fungi. 

For both datasets, limits in our current understanding of the ecological role specific taxa limits our ability to interpret patterns in the distribution of taxa across urban gradients. Most fungal taxa that were identified to genus or species can be classified according to trophic guild (saprotroph, symbiotroph, pathotroph), though this classification sometimes relied on reports of where taxa were observed [e.g. _Valsonectria pulchella_ is found on wood of _Melia_ sp. @seifert1997]. However, information on niche, preferred pH, preferred soil strata, etc. was difficult to obtain for all but the most well-studied species. Further, our ability to identify parasitic fungi appears much more limited than our ability to detect saprotrophic or symbiotrophic species. Parasitic fungi were detected on only 15 of 19 sites where mushrooms were sampled and the number of SV reads for pathotrophic fungi is approximately 1/3 that of saprotrophic fungi.



### Drivers of species richness and community composition

Despite potential bias, analyses of the two different datasets are generally aligned when identifying drivers of species richness and community composition. 

#### Drivers of overall effective species richness {-}

No variables significantly explained variation in effective species richness for either NGS or mushroom collection datasets. These results do not support my hypothesis that site scale variables describing development actions and those describing woody debris availability would explain variation in species richness. 

These results also generally disagree with previous research elsewhere, though I found no comparable studies in the Pacific Northwest. For example, a study in Sydney, Australia found a relationship between fungal species richness and measures of habitat fragmentation of remnant habitats, including patch size, perimeter to area ratio, and distance to other large reserves [@drinnan2005]. Habitat size was also significant in road medians and urban parks in Manhattan, though less than expected; distance between sites was not significant [@reese2015]. Results of a study in Milwaukee, WI, were ambiguous; while no consistent land-use effects were observed, increased urbanization appeared to decrease species richness [@turnquist2016].  

However, other research agreed with our results. A study in Coimbra, Portugal found that species richness and Shannon's diversity index in remnant forests and public gardens were similar [@barrico2018]. Overall, urbanization leads to complex landscape patterns, and measures of urbanization in other studies were inexact compared with the more specific measures used here [@alberti2008]. 

#### Drivers of saprotroph and symbiotroph effective species richness {-}

Based on previous research, I hypothesized that reduced fungal symbiotroph species richness would occur on more disturbed sites [@karpati2011; @cousins2003; @bainard2011; @jumpponen2005]. However, no variables, including neighborhood variables, soil variables, or those quantifying relative disturbance (e.g. impervious surface) were significant in explaining variation in saprotroph or symbiotroph effective species richness. These results do not provide support for my hypothesis.

More research addresses symbiotrophic species richness than saprotrophic species richness or richness of the overall fungal community. Many urban specific fungal studies did find significant drivers of symbiotrophic---particularly arbuscular mycorrhizal (AM) and ectomycorrhizal (EM) fungi---species richness. Researchers in New Jersey found that the root systems of trees grown in disturbed soils had lower EM fungal richness; they also concluded that this was likely to have little functional impact, as unless sites were severely disturbed (e.g. polluted, high pH, low organic matter, compacted) it was unlikely that symbiotrophic inoculum abundance would be significantly limited [@karpati2011]. In contrast, in Ontario, both AM and EM fungi colonized trees at lower rates in urban environments, suggesting that these areas have lower mycorrhizal fungal propagule abundance or infectivity [@bainard2011]. In Minnesota, researchers found a relationship between proximity to trees at the forest's edge and increased EM fungal species richness [@dickie2005]. However, given the relatively small size and the relatively high density of trees planted in office developments surveyed, my research likely did not sample as broad a range of tree density as did @dickie2005. Studies addressing saprotrophic species richness found that it was negatively correlated with canopy openness and positively correlated with available nitrogen concentrations [@newbound2012]. 

I found only one study that explicitly agreed with our results. The study in Manhattan found that AM fungi were not less species rich in street medians than urban parks, despite significant differences with site area for the whole population [@reese2015]. 

Differences between my study and other research may come from methodological differences. For example, while @drinnan2005 used similar fungal mushroom collection methods, @barrico2018 and @turnquist2016 used PCR gels instead of NGS techniques. Additionally, these studies examined remnant habitat and urban parks, not developed parcels, and used species richness instead of effective species richness (although these measures are generally fairly similar). Others looked at limited sub-sections of the fungal community; @karpati2011 examined EM fungal communities only on _Quercus rubra_ (northern red oak). 

Instead of urbanization or development gradients, stochastic processes, unmeasured or unknown processes, or that the observed taxa have highly variable environmental preferences may be responsible for variation in effective species richness. Stochastic assembly processes of random population fluctuations and dispersal explained beta-diversity in soil mite populations, and may for fungal populations as well [@caruso2017]. There may also be a temporal accumulation of fungal species richness. On green roofs, roof age was a significant explanatory variable for AM fungal abundance, while in urban parks, microbial biomass increased with park age [though species richness was not measured; @chaudhary2018; @francini2018].

Overall, the question of changes in species richness across environmental and/or disturbance gradients appears unsettled. However, I would argue that understanding the response of community composition and specific taxa is more important to our understanding of novel ecosystem process and changes in fungal community function. This is because species richness is an aggregate measure that tells us very little about the functions that fungal taxa are performing, while community composition and information about specific taxa may allow us to understand how function changes across environmental gradients in novel ecosystems.


### Drivers of community composition

Variables describing ground cover and soil chemistry for the NGS dataset, and site scale land management variables in both NGS and mushroom collection datasets, were consistently significant in explaining community composition. No detection variables for the collections dataset or neighborhood scale variables for either dataset were significant. Taken together, my results agree with other research and confirms my hypothesis that site scale variables are more important to fungal community composition than neighborhood scale variables [@newbound2012]. However, for all significant variables the effect size was consistently very small. 

Ground cover type (grass, mulch, unmanaged, or forest) was used to stratify sampling for the NGS dataset as ground cover influences fungal communities [Ammirati, personal communication; @trinder2009]. Controlling for ground cover type allowed me to test the effect of soil chemistry, land management, and neighborhood scale variables on community composition taking into account the different ground cover types present on each site. Here, my results agree with previous research; ground cover type significantly explained community composition for all subsets of the NGS dataset, and models containing ground cover alone were frequently the most parsimonious, see [Appendix III: Fungi](#CH9-fungiappendix)). No ground cover x site interaction effects were found, meaning that the ground cover effect does not vary depending on site. That ground cover type was so strongly associated with community composition implies that a land manager's choice of ground cover is an important action that may impact ecosystem service provision by fungi. In other words, covering a site with mulch vs. grass may be the strongest determinant of community composition and thus what taxa of fungi are present, and thus what functional traits are present (e.g. decomposition enxyme pathways, mycorrhizal capable of forming partnerships).

Soil chemistry variables (pH and carbon to nitrogen ratio) were consistently significant in explaining variation in community composition for the NGS dataset. Evidence for the mushroom collections dataset was weak with only C:N significant in explaining saprobic community composition, though because soil chemistry variables were calculated using weighted averages these results may not be accurate. Phosphorous concentration was not significant in either dataset. These results largely agree with previous research in Melbourne, Australia which found that soil chemical properties---including pH, nitrogen, carbon, and phosphorous---were significantly correlated with fungal community composition [@newbound2012]. My results also agree with research on agricultural land and natural landscapes finding changes in community composition over soil nutrient gradients, including pH and N gradients [@lilleskov2002; @treseder2004; @hogberg2007]. This result suggests that soil nutrients, regardless of source (fertilizer, previous land use, etc.) influence fungal community composition.

Site scale variables describing maintenance activity (e.g. detritus cleanup, mulching, irrigation, herbicide or fertilizer application, mushroom weeding, dead wood abundance) were not significant for either NGS or mushroom datasets. These results are surprising and contradict my hypothesis that fungal communities, including for saprotrophic and symbiotic communities, would be associated with dead wood abundance. Previous research on agricultural land found that fertilizer addition, particularly the addition of nitrogen, alters fungal communities [e.g. @lehto2011, @treseder2004], and results from this research found C:N ratio a significant explanatory variable for community composition. One possible explanation is that other drivers of soil chemistry (e.g. previous land use) are more important than the amounts of fertilizer added, particularly as reported frequency and application amounts were relatively small. Another probable explanation is that reported behavior and actual behavior differ. I interviewed landscaping professionals that coordinated---but did not perform---landscaping labor. This may also account for the discrepancy between stated mushroom removal and lack of impact on mushroom collections data, though field experience tells me that most people don't notice mushrooms unless it is an _Amanita muscaria_ or similarly large and bright colored.  

Site scale variables describing aspects of landscaping and development choices (e.g. presence of stands predating development, % impervious on site, median height of dominant Douglas fir, density of native conifers or native shrubs, and shrub effective species richness, tree and shrub communities) were significant explanatory variables for explaining fungal community composition for NGS or mushroom datasets (see Table \@ref(fig:CH4-fungi-var-summary)). 

Overall, native vegetation on site influences fungal community composition, while the importance of other development/land use history variables is uncertain. Findings that native plants are important agree with research in Phoenix, AZ, which found the presence of native plants influenced species composition of AM fungal communities, and unvegetated sites lacked AM fungal spores [@cousins2003]. Similar results were observed in the EM fungal community; in Kansas comparisons of communities on native and non-native ornamental host roots differed significantly even though species richness and diversity did not [@lothamer2014]. Surprisingly, dead wood did not explain variation in community composition. This finding disagrees with previous research using other methods where rare and endangered fungi were found to be more abundant with increasing amounts of dead wood [@norden2013]. 

Comparisons with other studies are difficult for variables like % impervious on site or presence of stands predating development, as measures of 'degree of urbanization' is imprecise in urban fungal literature. For example, researchers in Wisconsin found that neither degree of urbanization nor intensity of landscape management altered fungal community composition, however both of these were characterized only by land use, relative age (new and old residential neighborhoods, street terraces), and a qualitative intensity designation [intense/ minimal; @turnquist2016]. Research in Manhattan also found that land use did not impact fungal community composition [@reese2015]. These studies may have found different patterns of fungal community composition than my research--or it may be a matter of different uses and measures of 'urban' [@mcintyre2000urban]. Future studies should use more precise and ecologically relevant definitions; land use (e.g. 'residential') is remarkably heterogenous, and qualitative measures cannot be easily replicated.

No socio-economic variables were significant. This agrees with other studies, though they used different metrics for neighborhood socio-economic variables [including ratio of people density to urban land cover, neighborhood patch configuration; @newbound2012], and with my research on birds and vegetation in office developments.

As with univariate analysis, lack of consistency in collection methods is a barrier to comparing my research with previous research. Collection methodologies include NGS soil samples (this study), counting spores in soil samples [@cousins2003], sequencing mycorrhizal root fungal communities [@lothamer2014], and mushroom collection surveys [@drinnan2005; this research]. More consistent methodologies, including comparing different methodologies in laboratory and real world settings would aid our understanding of urban fungal communities.



### Response of fungal taxa to environmental gradients

This study marks the first use of TITAN in urban fungal research. Comparisons with fungal NGS studies performed in other ecosystems suggests that the proportion of SVs observed with significant p-values appears to compare favorably with other studies [@chariton2015; @jassey2017], though cutoffs for pure and reliable taxa criterion for these studies were not reported.

Correlation indices and TITAN produced consistent results. For both analyses and datasets no taxa were associated with the "less disturbed" end of one environmental gradient and the "more disturbed" end of another. For example, in the mushroom collections dataset, genera associated with "native" tree vegetation clusters (e.g. _Lycoperdon_ or _Arrhenia_) were also associated with the "less disturbed" ends of the native conifer density land management environmental gradient, and vice versa (e.g. _Cyclocybe_). Additionally, multiple genera, including _Xerocomellus_ and _Amanita_, were identified as z+ taxa in TITAN for both datasets. This provides additional support for both analyses, as two datasets measuring different portions of the fungal community resulted in similar statistical outcomes.

However, there was within-genus variation for the NGS dataset. A SV identified as _Suillus luteus_ was consistently associated with the "more disturbed" end of environmental gradients, including higher percent impervious surface and lower native confer density. On the other hand, an SV identified as _Suillus caerulescens_ was associated with the "less disturbed" end of environmental gradients. This contrast highlights the potential for within genera variability, and suggests that aggregating by genus for collections data---while practical and necessary for identification accuracy purposes--may obscure important within-genus patterns. It also highlights why aggregating SVs on genus would be unwise.

Evidence for community thresholds are present for many, but not all, variables. Both NGS and mushroom datasets suggest a community threshold at approximately 70% percent impervious surface on site. This estimate may be high due to the low number of sites sampled with impervious surface between 60-70% and how TITAN calculated change points[@king2014]. In comparison, research in Austin, TX found that most taxa change points were observed between 4-24% watershed impervious cover [@king2016]. While measuring neighborhood scale, this result suggests that sampling additional sites with < 50% impervious cover may reveal additional community thresholds--though sites with this little impervious surface are very rare.

A ecological community threshold was evident between 3.7-4.2 shrub effective species richness. There is evidence for two potential ecological community thresholds for native shrub density (shrubs per acre; at 20-45 and 100 shrubs/acre) and evidence for ecological community thresholds was also present for native conifer density (trees/acre). For both of these gradients, however, additional sampling of sites with greater than 50 shrubs per acre and greater than 25 trees/acre would help narrow the distribution of bootstrap replicates.  For these variables, both NGS and mushroom collection datasets had similar threshold estimation.

There is some evidence for an ecological community threshold for soil pH in the NGS dataset. Multiple negative (z-) and positive (z+) indicator taxa have change points near ph = 5.5; however these make up a relatively small proportion of pure and reliable taxa. It seems more likely based on change points and filtered sum(z) plots that there is a steady replacement of acid-loving taxa with those preferring more neutral pH between pH 5-6. pH of sites was well sampled between 4.5-6.5, however more basic sites were not observed. This is not surprising, as soils in the Pacific Northwest tend to be more acidic in part due to water transport of base cations and mulch and conifer needles are generally though acidic [@chalkerscott2007]. Similar studies took place in significantly more basic environments, so results can't be compared [@chariton2015].

Gradual community turnovers instead of clear ecological community thresholds were suggested for the remaining land management variables (median dominant Douglas-fir height, a proxy for tree age) and soil variables (C:N). Comparable studies did not measure median dominant canopy, and generally used nitrogen concentrations instead of C:N so it is difficult to evaluate how the results compare [@chariton2015].

While results from these analyses are generally interpretable and consistent, more questions remain. First, the majority of sequence variants in the NGS dataset with substrate ecological preferences (mulch, grass, unmanaged, forest) were associated with the 'unmanaged' habitat. Results from PERMANOVA and beta dispersion tests of ground cover suggest the 'unmanaged' ground cover habitat differs in both multivariate location and spread. Whether these results support each other--that is, that the unmanaged habitat is truly unique in some way--or are the result of a measurement error is unknown and additional study is required. Second, there are few additional studies to compare these results with. Finally, patchy site distribution for multiple environmental gradients and poor sampling of the entire (potentially very long) gradient means that we cannot dismiss the potential presence of thresholds [@king2014].

### Overall Response of Fungal Communities on Office Developments

The three methods (univariate regression, PERMANOVA, and TITAN) are complimentary and allow examination of multiple aspects of the fungal community. Overall, I conclude that many genera/SVs are adaptable to environmental gradients in novel ecosystems, distributed across the landscape stochastically, and/or distributed in response to patterns and processes not measured in this study. 

However, there are some genera/SVs with a strong response to some of the environmental gradients measured here arising from novel ecosystem processes, including soil chemistry (pH, C:N), ground cover, and development and land management (presence of stands predating development, % impervious surface on site, median height of dominant Douglas-fir, density of native conifers and shrubs, shrub effective species richness, and tree and shrub communities). Site scale variations in habitat are likely more important than neighborhood scale variations.

Fungi may be more resilient to disturbance than previously thought. For example, disturbance did not significantly alter AM fungal community composition or species richness in grasslands ecosystems [@lekberg2011].

Stochastic assembly processes are known to drive diversity in other organisms. Birth, death, colonization, and extinction (ecological drift processes) may strongly influence diversity and species composition, particularly in the absence of strong drivers such as drought [@chase2007]. Stochastic assembly processes explained beta-diversity in soil mite populations [@caruso2017] and may for fungi as well [e.g. @lekberg2011].  Other researchers have found little overlap in community composition between different sites in natural systems [@karpati2011].

Potential unmeasured environmental gradients including other soil characteristics, the specific time since disturbance at sub-site scales, or an unknown soil legacy effect from previous land use. Future research on urban fungi should collect data on other soil variables, including organic content [@diniandreote2016]. Additionally, future research must be sure to sample the entire gradient; some variables measured in this study may not be significant simply because I could not sample their entire range [@king2014]. 

There may also be a temporal accumulation of fungal species richness. While building age was not found significant for explaining variation in species richness or community composition, disturbance at sub-site scales as from significant landscaping changes may alter soil age at a sub-site scale. On green roofs, roof age was a significant explanatory variable for AM fungal abundance, while in urban parks, microbial biomass increased with park age [though species richness was not measured; @chaudhary2018; @francini2018]. However, not all studies agree [@diniandreote2016].

Other researchers have attributed observed differences in EMF community makeup to differences in individual site use histories [@karpati2011]. Studies in Arizona suggest that while AM fungi species are likely able to persist in the urban environment or immigrate from surrounding habitat, disturbance and agriculture impact AM fungi community structure and these results persist long-term [@cousins2003]. This may be the case with office developments as well, as multiple cycles of development and redevelopment, and significant landscaping changes were observed on sites in this study. For example, three sites in this study had previously been part of an airport strip built in the mid-1940s before it was decommissioned and redeveloped beginning in the mid-1980s. Other sites in the study were developed from single-family residential properties or out of second growth forest. Few good quantitative measures of previous land use exist; possible avenues of future research include digitizing and quantifying canopy cover and percent impervious from old aerial maps. 


### Policy implications and guidance for land managers

The results of this research alone should not form the basis for regulatory policy [sensu @king2014]. However, patterns found in this research combined with previous research and ecological theory provide some policy implications and guidance for city and land managers. 

Overall, the best approach for city and land managers to take is to follow the precautionary principle[^precautionary]. Practically, this would mean writing policy at the city level and acting at the parcel level to avoid unnecessary impacts on the existing fungal community and preventing infection vectors for pathogenic and invasive non-native fungi.

Choices made during development are likely important for a subset of the fungal community. These include tree and existing vegetation preservation, soil disturbance during clearing and grading phases, and impervious surface installation. Tree and other vegetation preservation influence host plant availability which in turn alters fungal community composition [@cousins2003; @lothamer2014; this research]. Preserving trees and existing vegetation also preserves nearby soil, creating reserves for fungi to recolonize the site after development. Ideally, cities would implement tree protection policies which are ecologically meaningful, easily complied with, costly to violate, and well enforced---however most city codes are ineffective and/or include perverse incentives[^seattleanaly]. Policies reducing soil disturbance during clearing and grading will likely reduce change in fungal community composition. Additionally, percent impervious on site was consistently a significant variable explaining variation in mushroom collection communities. Policy impacting percent impervious surface on site, including parking lot requirements in zoning code will impact potential habitat area. Impervious surface policy will also impact habitat fragmentation, soil properties, and runoff pollution, which may also be mechanisms of fungal community change [@davis2010].

Choices made after development, including landscaping choices and ground cover choices, are also likely important for a subset of the fungal community. Key policies actions include substituting natives for non-natives on tree planting lists, which frequently feature deciduous species from eastern North America, Europe, and Asia [@cousins2003; @lothamer2014; this research]. For land managers, choosing to retain downed logs and snags---though maybe not add them artificially without further local research---may be beneficial for fungal conservation [@juutilainen2016; @komonen2014; @dove2015; though see @runnel2013]. Ground cover was the most consistently important variable explaining variation in fungal community composition in PERMANOVA results. Ground cover also drives other maintenance actions like irrigation and fertilizer addition, which were not significant here but have been in other studies.
Choices made after development, including landscape design choices and ground cover choices, are also likely important for a subset of the fungal community. Key policies actions include substituting natives for non-natives on tree planting lists, which frequently feature deciduous species from eastern North America, Europe, and Asia, and encouraging native shrub planting [@cousins2003; @lothamer2014; this research]. Ground cover was the most consistently important variable explaining variation in fungal community composition in PERMANOVA results. Ground cover also drives other maintenance actions like irrigation and fertilizer addition, which were not significant here but have been in other studies. More research is needed to see if choosing to retaining downed logs and snags may be beneficial for fungal conservation [@juutilainen2016; @komonen2014; @dove2015; though see @runnel2013]. 

Finally, reducing potential vectors for high-risk pathogenic fungi and non-native invasive fungi is prudent. In the Puget Sound region, logging activities between 1880-1920 likely contributed to the spread of laminated root rot (caused by _Phellinus sulphurascens_ in Douglas fir) and other root pathogens. Development activities have the potential to do the same via transport of topsoil containing infected root tissue. Encouraging construction hygiene, including reducing direct damage to tree trunks and roots by ensuring that dripline protection is sufficiently large, and reducing movement of soil with potentially infected woody material between sites are important measures. As we know little about which species are non-native and possibly invasive, it is prudent to pay attention to and research potential vectors. Potential vectors include mulch, fungal inoculation at nurseries, and soil movement (Ammirati, personal communication).



[^precautionary]: Following the 1992 Rio Declaration on Environment and Development, "Where there are threats of  serious or irreversible damage, lack of full scientific certainty shall not be used as a reason for postponing cost-effective measures to prevent environmental degradation." 

[^seattleanaly]: Seattle recently completed an analysis of its own tree protection policies entitled Tree Regulations Research Project Phase II and available online: https://www.documentcloud.org/documents/4114063-Final-Report-Tree-Regulation-Research-Project.html.


### Further research

Far less is known about urban fungi than other taxa from systematic study, and there are fewer bases for comparison. As a result, this work on fungi is more exploratory than the concurrent vegetation and bird research (Chapters 2 and 3, respectively). This study identifies multiple aspects of novel ecosystem processes influencing fungal community composition. However, small effects size suggests more research is needed to determine if these are statistical artefacts or real patterns, and if other yet unknown drivers exist (e.g. historic land use, above). Multiple complimentary plans of study are needed to better understand urban fungi, and there are many observational, longitudinal, and manipulation studies that should be done. Access to private property to conduct these experiments remains a concern and constraint for future research [@dyson2018].

The interaction between fungal community composition and taxa responses to urban environmental gradients must be further explored, including those gradients studied here, land use history, more complex soil and water chemistry including non-point source pollution runoff, among others. Variables measured here using interview responses instead of direct measurement (e.g. fertilizer application, irrigation) need to be more accurately measured and tested. 

Sampling along environmental gradients is challenging. The entire environmental gradient must be measured for analyses (e.g. TITAN) to make meaningful conclusions [@king2014]. However, the extent of the gradient for many variables isn't known until samples are collected (e.g. with soil chemistry). Large numbers of samples are therefore required. In this research, I stratified by rough vegetation categories to capture the entire vegetation gradient. If I had taken a random sample of office developments, I would have ended up with a subset of many of the most important site scale variables including dead wood abundance and presence of stands predating development, and many of these variables would not have been significant in PERMANOVA or revealed synchronous change points indicative of ecological fungal community thresholds. 
Sampling along environmental gradients is challenging. The entire environmental gradient must be measured for analyses (e.g. TITAN) to make meaningful conclusions [@king2014]. However, the extent of the gradient for many variables isn't known until samples are collected (e.g. with soil chemistry). Large numbers of samples are therefore required. In this research, I stratified by rough vegetation categories to capture the entire vegetation gradient. If I had taken a random sample of office developments, I would have ended up with a subset of many important site scale variables including native conifer abundance and presence of stands predating development, and many of these variables would not have been significant in PERMANOVA or revealed synchronous change points indicative of ecological fungal community thresholds. 

To better understand ecosystem services and disservices provided by urban fungi and how provision is impacted by novel ecosystem process, direct tests of fungal function are needed. These include litter bag tests and soil enzyme activity studies for examining decomposition, and mycorrhizal colonization experiments and root tip fungal sequencing for mycorrhizal communities [@jassey2017]. More research examining the relationship between mycorrhizal species and their hosts, including rates of carbon and nutrient transfer, and whether these rates change along environmental gradients dictated by novel ecosystem processes. Pathogenic fungal species provide significant ecosystem disservices in urban areas, including increased risk of windthrow from root rot fungi. Previous research on pathogenic fungi has generally been focused on their impact on forestry, where losses from pathogenic fungi are measured in board feet, rather than property damage or potential loss of life. The higher risk profile of urban areas increases the importance of research into mapping urban hazards, early hazard tree detection, and tree susceptibility.

Further, many basic questions about fungal communities in urban areas remain unanswered. For example, not much is known about fungal dispersal---either wind or animal or insect-borne---in urban areas. We also don't know how much spores contribute to fungal dispersal and colonization, though it likely varies significantly by taxa (e.g. _Armillaria_ vs. _Phellinus_).

For these and other questions, a long term plan of study is also required. We, in conjunction with other urban fungal researchers in other areas of the country and the world, need to accumulate a large dataset of fungi over many years to determine preferred habitat for fungi [@jumpponen2005] and build up a more accurate picture of key processes including dispersal (Ammirati, personal communication). Ongoing data collection should include specimen collection, culturing, and sequencing to continue adding to the UNITE database and the broader fungal taxonomy. This research found a new forma, and there are certainly additional species and forma to be found in urban areas.
For these and other questions, a long term plan of study is also required. We, in conjunction with other urban fungal researchers in other areas of the country and the world, need to accumulate a large dataset of fungi over many years to determine preferred habitat for fungi [@jumpponen2005] and build up a more accurate picture of key processes including dispersal (Ammirati, personal communication). Ongoing data collection should include mushroom collection, culturing, and sequencing to continue adding to the UNITE database and the broader fungal taxonomy. This research found a new forma, and there are certainly additional species and forma to be found in urban areas.

Further, these types of basic fungal research are necessary to determine ranges for species and whether species are native or introduced. Range information for many fungal species is very patchy, often including few specimens from very different locations. In this research, I found species known from other continents--but without additional observations it is impossible to know if they widespread or introduced species. Both mushroom collections and NGS/UNITE databases are impacted by this gap in our knowledge. Many of the range-based ecological approaches taken for granted in mammalogy, ornithology, etymology, and botany cannot be meaningfully applied in mycology. 

Methods for analyses using fungal DNA ranged widely, and NGS techniques have improved substantially in the past 10 years. It would be worth taking NGS data and testing how different processing pipelines (raw reads to OTU vs SVs with dada2) and aggregation approaches (leave at OTU/SV vs. aggregate by genus) change end results. Additionally, using appropriate multivariate techniques including PERMANOVA and TITAN are likely to provide more insight into changes in fungal community composition and what taxa are driving community thresholds. These insights are obscured by univariate techniques [@king2014; this research]. If there are differences in outcomes based on processing pipelines, aggregation approaches, or if studies have relied on univariate approaches, there is convincing argument to be made for reanalyzing older datasets collected under well-conceived sampling designs employing improved statistical techniques [@callahan2016]. 

These questions only scratch the surface of how much research remains to be done on urban fungi. Continuing research to determine novel ecosystem processes which alter the distribution and function of fungi in urban areas is necessary to conserve fungi in cities [@newbound2010]. Interdisciplinary research with the social sciences into city and land manager motivations and preferences may also provide important conservation insight but is outside the scope of this study.





## Conclusion

<<<<<<< HEAD
Novel ecosystem processes arise when developing, landscaping, and maintaining urban land modifies or replaces ecosystem processes in urban areas. These novel ecosystem processes alter soil and wood fungal habitats by altering vegetation communities, habitat area and fragmentation, and changing nutrient availability and soil chemistry. Using NGS and collected fungal specimens, I found that parcel scale maintenance actions that altered ground cover, soil chemistry and nutrient availability, native tree size and abundance, shrub richness and abundance, and percent impervious surface were strongly linked to observed fungal community composition, but not effective species richness. Many of these environmental gradients were associated with ecosystem community thresholds. 
Novel ecosystem processes are influenced by the actions of humans and other species. Human actions include development, landscape design, and maintaining urban land; the outcome of these actions alters fungal habitats in novel ecosystems by altering vegetation communities, habitat area and fragmentation, and changing nutrient availability and soil chemistry [@newbound2010; @newbound2012; @batten2006]. Using NGS and collected mushroom specimens, I found that parcel scale development and landscape design actions that altered ground cover, soil chemistry and nutrient availability, native tree size and abundance, shrub richness and abundance, tree and shrub community composition, and percent impervious surface were strongly linked to observed fungal community composition, but not effective species richness. There were also ecosystem community thresholds associated with many of the environmental gradients resulting from development and landscape design actions.

Overall, many fungal taxa on office developments in Redmond and Bellevue, WA are adaptable to novel gradients, distributed across the landscape stochastically, and/or distributed in response to patterns and processes not measured in this study. However, a group of fungal taxa have a strong response to these novel environmental gradients resulting in part from human action. The majority of the taxa in this group are positively associated with "less disturbed" ends of the gradient---e.g. greater native conifer or shrub density or less impervious surface---and exhibit change points that contribute to evidence for ecosystem community thresholds. These thresholds suggest changes in ecosystem function provided by fungi across the gradient, however we do not have enough information about individual fungal taxon function provision to draw firm conclusions. Future urban fungal research should explicitly measure ecosystem process across the gradients identified here.

The paucity of urban fungal research generally makes this research more exploratory in nature than other chapters in this dissertation. How generalizable these results are is uncertain and possibly variable. I suggest that that ground cover (mulch, grass, unmanaged, forest) matters across the urban landscape. If management intensity varies between land uses [e.g. residential property nutrient input may be higher; @fissore2012] I expect more fungal taxa to be impacted by environmental gradients resulting from ecosystem process. 

Based on these results and that comparatively little is known about the response of urban fungi to novel ecosystem processes, the best approach for city and land managers to take is to follow the precautionary principle. Policy and actions that avoid unnecessary impacts on pre-existing fungal communities, encouraging native tree and shrub communities, and prevent vectors for pathogenic and invasive non-native fungi are prudent.
